--- p51.asm	2025-05-27 08:09:41.997116338 +0200
+++ pascal51.asm	2025-05-27 08:09:42.003116339 +0200
@@ -2,11 +2,12 @@
 ; command line: z80dasm -b p51.block -g 0x200 -l -S p51.sym -s p51.symout -u -o pascal51.asm pascal51_kcc.bin
 
 	org	00200h
-iobuf+11:	equ 0x008b
-iobuf+16:	equ 0x0090
-iobuf+17:	equ 0x0091
-iobuf+19:	equ 0x0093
-iobuf+20:	equ 0x0094
+iobuf:	equ 0x0080
+;iobuf+11:	equ 0x008b
+;iobuf+16:	equ 0x0090
+;iobuf+17:	equ 0x0091
+;iobuf+19:	equ 0x0093
+;iobuf+20:	equ 0x0094
 CAOS_ARG1:	equ 0xb782
 CAOS_ARG2:	equ 0xb784
 CAOS_ARG3:	equ 0xb786
@@ -19,63 +20,55 @@
 CAOS_VERT:	equ 0xb7d5
 CAOS_FARB:	equ 0xb7d6
 SUBALT:	equ 0xb7fe
-PXSASCI:	equ 0xbc00
-PXSASCI+1:	equ 0xbc01
-PXASCI:	equ 0xbc08
-PXTAPE:	equ 0xbc0f
-PXTAPE+1:	equ 0xbc10
-PXDISK:	equ 0xbc20
-PXDISK+1:	equ 0xbc21
-ISRO:	equ 0xbc2f
-PXNxtBlock:	equ 0xbc47
-PXSendCtl:	equ 0xbc5c
-MBO:	equ 0xbc80
-CSRO:	equ 0xbc96
-ISRI:	equ 0xbc9e
-PXRecvBlock:	equ 0xbcb2
-MBI:	equ 0xbcd0
-CSRI:	equ 0xbce9
-SUBNEU:	equ 0xbcee
-SUBNEU+2:	equ 0xbcf0
-SUBNEU+10:	equ 0xbcf8
-SUBNEU+16:	equ 0xbcfe
+;;PXSASCI:	equ 0xbc00
+;PXSASCI+1:	equ 0xbc01
+;;PXASCI:	equ 0xbc08
+;;PXTAPE:	equ 0xbc0f
+;PXTAPE+1:	equ 0xbc10
+;;PXDISK:	equ 0xbc20
+;PXDISK+1:	equ 0xbc21
+;;ISRO:	equ 0xbc2f
+;;PXNxtBlock:	equ 0xbc47
+;;PXSendCtl:	equ 0xbc5c
+;;MBO:	equ 0xbc80
+;;CSRO:	equ 0xbc96
+;;ISRI:	equ 0xbc9e
+;;PXRecvBlock:	equ 0xbcb2
+;;MBI:	equ 0xbcd0
+;;CSRI:	equ 0xbce9
+;;SUBNEU:	equ 0xbcee
+;SUBNEU+2:	equ 0xbcf0
+;SUBNEU+10:	equ 0xbcf8
+;SUBNEU+16:	equ 0xbcfe
 PV1:	equ 0xf003
 
+IRM_ON	MACRO
+	in a,(088h)
+	set 5,a
+	set 2,a
+	out (088h),a
+	ENDM
+	
+IRM_OFF	MACRO
+	in a,(088h)
+	res 5,a
+	res 2,a
+	out (088h),a
+	ENDM
+	
 PasPrgMenuHdr:
 	defw 00000h
 PasPrgMenuName:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l0208h:
-	defb 000h
-	defb 000h
+	defs 8
 	defb 000h
 	jp PasPrgStart
 MenuPRec:
 	defw 07f7fh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 052h
-	defb 045h
-	defb 043h
-	defb 000h
+	defb 'PASREC',000h
 	jp Recall
 MenuPEnt:
 	defw 07f7fh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 045h
-	defb 04eh
-	defb 054h
-	defb 052h
-	defb 059h
-	defb 000h
+	defb 'PASENTRY',000h
 	jp Entry
 DoNopRET:
 	nop	
@@ -92,10 +85,7 @@
 	ld ix,(caos_ix)
 	ld (pascal_sp),sp
 	ld (tmp_reg_a),a
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld a,e	
 	ld (OSPrc),a
 	ld a,(tmp_reg_a)
@@ -107,10 +97,7 @@
 	di	
 	ld (caos_sp),sp
 	ld (tmp_reg_a),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld a,(tmp_reg_a)
 	ld sp,(pascal_sp)
 	pop hl	
@@ -132,15 +119,9 @@
 	push de	
 	push ix
 	ld ix,(caos_ix)
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	res 0,(ix+008h)
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ei	
 	ld a,002h
 	ld e,014h
@@ -151,33 +132,21 @@
 	call CCaos
 	cp 080h
 	jr c,CAOS_KBD
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	xor a	
 	ld (ix+00dh),a
 	res 0,(ix+008h)
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	jr GetKeyTest
 CAOS_KBD:
 	ld e,004h
 	call CCaos
 	cp 080h
 	jr nc,CAOS_KBD
-	ld e,a	
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	ld e,a
+	IRM_ON
 	res 0,(ix+008h)
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld a,e	
 	pop ix
 	pop de	
@@ -205,7 +174,6 @@
 	ld (bin_end),de
 	ld (iobuf+19),de
 	ld a,002h
-l0314h:
 	ld (iobuf+16),a
 	ld a,(fileExt)
 	cp 043h
@@ -235,41 +203,28 @@
 C_ISRO:
 	ld hl,fileName
 	ld bc,0000bh
-	ld de,00080h
+	ld de,iobuf
 	ldir
-	ld hl,00080h
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	ld hl,iobuf
+	IRM_ON
 	ld (ix+005h),l
 	ld (ix+006h),h
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
-	ld bc,l1f40h
+	IRM_OFF
+	ld bc,01f40h
 	ld e,008h
 	call CCaos
 SaveBlock:
 	ld hl,(SaveBlockAdr)
-	ld de,00080h
+	ld de,iobuf
 	ld bc,00080h
 	ldir
 	ld (SaveBlockAdr),hl
 	ld bc,000a0h
 	ld e,001h
 	call CCaos
-	in a,(088h)
-l0390h:
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld e,(ix+002h)
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld a,e	
 	call PrByteHex
 	call PrSpace
@@ -297,38 +252,26 @@
 	ld ix,(caos_ix)
 C_ISRI:
 	call TestBreak
-	ld hl,00080h
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	ld hl,iobuf
+	IRM_ON
 	set 0,(ix+007h)
 	ld (ix+005h),l
 	ld (ix+006h),h
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld e,00ah
 	call CCaos
 	jr c,RetryISRI
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld e,(ix+002h)
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld a,e	
 ChkForBlock1:
 	cp 001h
 	jp nz,RetryISRI
 	inc a	
 	ld (nextBlkNum),a
-	ld de,00080h
-	ld bc,l0b00h
+	ld de,iobuf
+	ld bc,00b00h
 	ld hl,fileName
 CmpFNamChr:
 	ld a,(de)	
@@ -366,15 +309,9 @@
 	ld e,005h
 	call CCaos
 	jr c,C_MBI
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld e,(ix+002h)
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld a,e	
 	ld hl,nextBlkNum
 	cp 0ffh
@@ -399,14 +336,14 @@
 	jr c,C_CSRI
 	ld (SaveBlockAdr),hl
 	ld bc,00080h
-	ld hl,00080h
+	ld hl,iobuf
 	ld de,(srcAddr_LoadSave)
 	ldir
 	ld (srcAddr_LoadSave),de
 	jr C_MBI
 C_CSRI:
 	ld bc,(SaveBlockAdr)
-	ld hl,00080h
+	ld hl,iobuf
 	ld de,(srcAddr_LoadSave)
 	ldir
 	ld e,00bh
@@ -421,7 +358,6 @@
 	jp CCaos
 CAOS_UOT1:
 	push af	
-l04dch:
 	ld a,c	
 	push de	
 	ld e,002h
@@ -433,16 +369,10 @@
 	ld a,(l178fh)
 	ld h,a	
 	ld a,(l178dh)
-	ld l,a	
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	ld l,a
+	IRM_ON
 	ld (CAOS_CURSO),hl
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ret	
 sub_0502h:
 	ld a,(l178fh)
@@ -454,23 +384,14 @@
 	rlca	
 	rlca	
 	or l	
-	ld l,a	
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	ld l,a
+	IRM_ON
 	ld a,l	
 	ld (CAOS_COLOR),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ret	
 sub_0527h:
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld hl,(l178dh)
 	ld (CAOS_HOR),hl
 	ld a,(l178fh)
@@ -478,11 +399,7 @@
 	ld a,(CAOS_COLOR)
 	and 0f8h
 	ld (CAOS_FARB),a
-	in a,(088h)
-l0545h:
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 sub_054bh:
 	push de	
 	ld e,030h
@@ -495,10 +412,7 @@
 	ld h,000h
 	jr z,l0588h
 	ld (l1791h),hl
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld hl,(l178dh)
 	ld (CAOS_HOR),hl
 	ld a,(l178fh)
@@ -506,10 +420,7 @@
 	ld a,(l1791h)
 	and 0f8h
 	ld (CAOS_FARB),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	call sub_054bh
 	ld hl,(l1791h)
 l0588h:
@@ -526,10 +437,7 @@
 	ld h,000h
 	jr z,l05c5h
 	ld (l1791h),hl
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld hl,(l178dh)
 	ld (CAOS_HOR),hl
 	ld a,(l178fh)
@@ -537,10 +445,7 @@
 	ld a,(l1791h)
 	and 0f8h
 	ld (CAOS_FARB),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	call sub_054bh
 	ld a,001h
 	ret	
@@ -548,28 +453,19 @@
 	xor a	
 	ret	
 sub_05c7h:
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld hl,(l178dh)
 	ld (CAOS_HOR),hl
 	ld a,(l178fh)
 	ld (CAOS_VERT),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	push de	
 	ld e,02fh
 	call CCaos
 	pop de	
 	ret	
 sub_05ebh:
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld hl,(l178dh)
 	ld (CAOS_ARG1),hl
 	ld hl,(l178fh)
@@ -581,20 +477,14 @@
 	ld a,(CAOS_COLOR)
 	and 0f8h
 	ld (CAOS_FARB),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	push de	
 	ld e,03eh
 	call CCaos
 	pop de	
 	ret	
 sub_0623h:
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld hl,(l178dh)
 	ld (CAOS_ARG1),hl
 	ld hl,(l178fh)
@@ -604,44 +494,26 @@
 	ld a,(CAOS_COLOR)
 	and 0f8h
 	ld (CAOS_FARB),a
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	push de	
 	ld e,03fh
 	call CCaos
 	pop de	
 	ret	
 sub_0655h:
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld a,(l178dh)
-	ld (hl),a	
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	ld (hl),a
+	IRM_OFF
 	ret	
 sub_066ah:
-	in a,(088h)
-	set 5,a
-	set 2,a
-	out (088h),a
+	IRM_ON
 	ld l,(hl)	
 	ld h,000h
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ret	
 GetRAMEnd:
-	in a,(088h)
-	res 5,a
-	res 2,a
-	out (088h),a
+	IRM_OFF
 	ld hl,06000h
 l0689h:
 	ld e,(hl)	
@@ -724,18 +596,9 @@
 JCompileToRuntimeEnd:
 	jp SetBinStartToRuntimeEnd
 fileName:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h,000h,000h,000h,000h,000h
 fileExt:
-	defb 000h
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h
 	defb 000h
 l0704h:
 	defw 00000h
@@ -847,13 +710,9 @@
 	ld hl,fileExtCOM
 	jr SetFileExt
 fileExtCOM:
-	defb 043h
-	defb 04fh
-	defb 04dh
+	defb 'COM'
 fileExtPAS:
-	defb 050h
-	defb 041h
-	defb 053h
+	defb 'PAS'
 SaveSrcFile:
 	push ix
 	push iy
@@ -1054,21 +913,12 @@
 
 ; BLOCK 'TFalse' (start 0x08d4 end 0x08da)
 TFalse_start:
-	defb 046h
-	defb 041h
-	defb 04ch
-	defb 053h
-	defb 045h
-	defb 000h
+	defb 'FALSE',000h
 tTrue:
 
 ; BLOCK 'TTrue' (start 0x08da end 0x08df)
 TTrue_start:
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 045h
-	defb 000h
+	defb 'TRUE',000h
 PrRAM:
 	ld de,T_RAM
 	jr PrErr2
@@ -1127,139 +977,25 @@
 ; BLOCK 'TErrMsg' (start 0x0934 end 0x09b1)
 TErrMsg_start:
 	defb 00dh
-	defb 048h
-	defb 061h
-	defb 06ch
-	defb 074h
+	defb 'Halt'
 T_beiPC:
-	defb 020h
-	defb 062h
-	defb 065h
-	defb 069h
-	defb 020h
-	defb 050h
-	defb 043h
-	defb 03dh
-	defb 000h
+	defb ' bei PC=',000h
 TUeber:
-	defb 055h
-	defb 065h
-	defb 062h
-	defb 065h
-	defb 072h
-	defb 06ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 000h
+	defb 'Ueberlauf',000h
 T_RAM:
-	defb 03eh
-	defb 03eh
-	defb 020h
-	defb 052h
-	defb 041h
-	defb 04dh
-	defb 000h
+	defb '>> RAM',000h
 TDiv0:
-	defb 02fh
-	defb 020h
-	defb 064h
-	defb 075h
-	defb 072h
-	defb 063h
-	defb 068h
-	defb 020h
-	defb 04eh
-	defb 075h
-	defb 06ch
-	defb 06ch
-	defb 000h
+	defb '/ durch Null',000h
 TIdxLow:
-	defb 049h
-	defb 06eh
-	defb 064h
-	defb 065h
-	defb 078h
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 06eh
-	defb 069h
-	defb 065h
-	defb 064h
-	defb 072h
-	defb 069h
-	defb 067h
-	defb 000h
+	defb 'Index zu niedrig',000h
 TIdxHigh:
-	defb 049h
-	defb 06eh
-	defb 064h
-	defb 065h
-	defb 078h
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 068h
-	defb 06fh
-	defb 063h
-	defb 068h
-	defb 000h
+	defb 'Index zu hoch',000h
 TMathErr:
-	defb 06dh
-	defb 061h
-	defb 074h
-	defb 068h
-	defb 065h
-	defb 06dh
-	defb 061h
-	defb 074h
-	defb 069h
-	defb 073h
-	defb 063h
-	defb 068h
-	defb 065h
-	defb 072h
-	defb 020h
-	defb 046h
-	defb 065h
-	defb 068h
-	defb 06ch
-	defb 065h
-	defb 072h
-	defb 000h
+	defb 'mathematischer Fehler',000h
 TNumTooBig:
-	defb 05ah
-	defb 061h
-	defb 068h
-	defb 06ch
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 067h
-	defb 072h
-	defb 06fh
-	defb 07eh
-	defb 000h
-	defb 000h
+	defb 'Zahl zu gro',07eh,000h,000h
 TNumExpected:
-	defb 05ah
-	defb 061h
-	defb 068h
-	defb 06ch
-	defb 020h
-	defb 065h
-	defb 072h
-	defb 077h
-	defb 061h
-	defb 072h
-	defb 074h
-	defb 065h
-	defb 074h
-	defb 000h
+	defb 'Zahl erwartet',000h
 SelListElem__:
 	ld d,ixh
 	ld e,ixl
@@ -1517,7 +1253,6 @@
 	jr LineEdLoop__
 TestCuD:
 	cp 010h
-l0b00h:
 	jr nz,TestEnter
 	call OutChr
 	jr LineEdNextLine__
@@ -2289,7 +2024,7 @@
 	ret c	
 	push hl	
 	push de	
-	ld bc,l5000h
+	ld bc,05000h
 	jr nz,l0f4ch
 	or a	
 	sbc hl,bc
@@ -2414,7 +2149,7 @@
 	call sub_0e45h
 	call sub_0ca1h
 l1022h:
-	ld bc,l5000h
+	ld bc,05000h
 	push bc	
 	ld b,003h
 	push bc	
@@ -2560,7 +2295,6 @@
 	ld a,045h
 	call OutChr
 	ld a,(l1797h)
-l1110h:
 	or a	
 	jp p,l111bh
 	neg
@@ -2612,7 +2346,7 @@
 	ld a,d	
 	or a	
 	jp p,l10bdh
-	ld hl,l5000h
+	ld hl,05000h
 	ld de,00300h
 	call RealMul__
 	push hl	
@@ -2645,15 +2379,8 @@
 	defb 0e2h
 	defb 053h
 realStrPart:
-	defb 020h
-	defb 030h
-	defb 02eh
-	defb 000h
-	defb 045h
-	defb 02bh
-	defb 030h
-	defb 030h
-	defb 000h
+	defb ' 0.',000h
+        defb 'E+00',000h
 Sqrt__:
 	ld a,h	
 	or a	
@@ -2843,24 +2570,7 @@
 	ld de,tExpErw
 	jp PrErr2
 tExpErw:
-	defb 045h
-	defb 078h
-	defb 070h
-	defb 06fh
-	defb 06eh
-	defb 065h
-	defb 06eh
-	defb 074h
-	defb 020h
-	defb 065h
-	defb 072h
-	defb 077h
-	defb 061h
-	defb 072h
-	defb 074h
-	defb 065h
-	defb 074h
-	defb 000h
+	defb 'Exponent erwartet',000h
 Frac__:
 	ld a,h	
 	or a	
@@ -2957,11 +2667,11 @@
 	push de	
 	ld bc,0c53fh
 	push bc	
-	ld bc,C_ISRI
+	ld bc,003d6h
 	push bc	
 	ld bc,063e7h
 	push bc	
-	ld bc,l04dch
+	ld bc,004dch
 	push bc	
 	call RealDiv__
 	call sub_0ca1h
@@ -3018,27 +2728,27 @@
 	ld a,d	
 	ld (l1797h),a
 	ld d,000h
-	ld bc,ParseSimEx1+1
+	ld bc,04073h
 	push bc	
-	ld bc,l02a1h
+	ld bc,002a1h
 	push bc	
 	ld bc,0c4d2h
 	push bc	
-	ld bc,l0545h
+	ld bc,00545h
 	push bc	
 	push hl	
 	push de	
-	ld bc,l5309h
-	push bc	
-	ld bc,l0390h
-	push bc	
+	ld bc,05309h
+	push bc
+	ld bc,00390h
+	push bc
 	ld bc,0c103h
 	push bc	
-	ld bc,l0314h
+	ld bc,00314h
 	push bc	
 	push hl	
 	push de	
-	ld bc,l41a3h
+	ld bc,041a3h
 	push bc	
 	ld bc,00189h
 	push bc	
@@ -3403,7 +3113,7 @@
 	push de	
 	add a,a	
 	add a,a	
-	ld hl,01665h
+	ld hl,xxTab3__-4
 	ld e,a	
 	ld d,000h
 	add hl,de	
@@ -3479,7 +3189,7 @@
 	call sub_0ca1h
 	push hl	
 	push de	
-	ld bc,l4305h
+	ld bc,04305h
 	push bc	
 	ld bc,0ff49h
 	push bc	
@@ -3548,7 +3258,7 @@
 	push bc	
 	push hl	
 	push de	
-	ld bc,l52aeh
+	ld bc,052aeh
 	push bc	
 	ld bc,003f5h
 	push bc	
@@ -3563,47 +3273,9 @@
 	call RealMul__
 	jp l150eh
 l1761h:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 8
 l1769h:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 32
 retAddr__:
 	defw 00000h
 l178bh:
@@ -3640,175 +3312,11 @@
 iBufCurChrAddr:
 	defw 00000h
 inputBuf:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 85
 RuntimeEnd:
 	nop	
 lineBuf:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l1819h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 80
 lineBufEnd:
 	defb 00dh
 stack_adr:
@@ -3820,22 +3328,12 @@
 	defw START_PASCAL
 RL_SETSYS:
 	defw 00000h
-	defb 053h
-	defb 045h
-	defb 054h
-	defb 053h
-	defb 059h
-	defb 0d3h
+	defb 'SETSY','S'+0x80
 	defb 006h
 	defw PaSetsys
 RL_GETSYS:
 	defw RL_SETSYS
-	defb 047h
-	defb 045h
-	defb 054h
-	defb 053h
-	defb 059h
-	defb 0d3h
+	defb 'GETSY','S'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3843,30 +3341,17 @@
 	defb 002h
 RL_PLOT:
 	defw RL_GETSYS
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'PLO','T'+0x80
 	defb 006h
 	defw l5174h
 RL_CLRPLOT:
 	defw RL_PLOT
-	defb 043h
-	defb 04ch
-	defb 052h
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'CLRPLO','T'+0x80
 	defb 006h
 	defw l51a0h
 RL_PTEST:
 	defw RL_CLRPLOT
-	defb 050h
-	defb 054h
-	defb 045h
-	defb 053h
-	defb 0d4h
+	defb 'PTES','T'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -3874,10 +3359,7 @@
 	defb 003h
 RL_GETC:
 	defw RL_PTEST
-	defb 047h
-	defb 045h
-	defb 054h
-	defb 0c3h
+	defb 'GET','C'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3885,48 +3367,27 @@
 	defb 003h
 RL_SETC:
 	defw RL_GETC
-	defb 053h
-	defb 045h
-	defb 054h
-	defb 0c3h
+	defb 'SET','C'+0x80
 	defb 006h
 	defw l5160h
 RL_LINEPLOT:
 	defw RL_SETC
-	defb 04ch
-	defb 049h
-	defb 04eh
-	defb 045h
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'LINEPLO','T'+0x80
 	defb 006h
 	defw l51b4h
 RL_CIRCLE:
 	defw RL_LINEPLOT
-	defb 043h
-	defb 049h
-	defb 052h
-	defb 043h
-	defb 04ch
-	defb 0c5h
+	defb 'CIRCL','E'+0x80
 	defb 006h
 	defw l51d0h
 RL_GOTOXY:
 	defw RL_CIRCLE
-	defb 047h
-	defb 04fh
-	defb 054h
-	defb 04fh
-	defb 058h
-	defb 0d9h
+	defb 'GOTOX','Y'+0x80
 	defb 006h
 	defw l5136h
 RL_PI:
 	defw RL_GOTOXY
-	defb 050h
-	defb 0c9h
+	defb 'P','I'+0x80
 	defb 001h
 	defb 002h
 	defb 000h
@@ -3936,21 +3397,12 @@
 	defb 064h
 RL_FRAC:
 	defw RL_PI
-	defb 046h
-	defb 052h
-	defb 041h
-	defb 0c3h
+	defb 'FRA','C'+0x80
 	defb 00bh
 	defw Frac__
 RL_READKBD:
 	defw RL_FRAC
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 044h
-	defb 04bh
-	defb 042h
-	defb 0c4h
+	defb 'READKB','D'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -3958,9 +3410,7 @@
 	defb 001h
 RL_SHR:
 	defw RL_READKBD
-	defb 053h
-	defb 048h
-	defb 0d2h
+	defb 'SH','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3968,9 +3418,7 @@
 	defb 003h
 RL_SHL:
 	defw RL_SHR
-	defb 053h
-	defb 048h
-	defb 0cch
+	defb 'SH','L'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3978,8 +3426,7 @@
 	defb 003h
 RL_LO:
 	defw RL_SHL
-	defb 04ch
-	defb 0cfh
+	defb 'L','O'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3987,8 +3434,7 @@
 	defb 002h
 RL_HI:
 	defw RL_LO
-	defb 048h
-	defb 0c9h
+	defb 'H','I'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3996,10 +3442,7 @@
 	defb 002h
 RL_SWAP:
 	defw RL_HI
-	defb 053h
-	defb 057h
-	defb 041h
-	defb 0d0h
+	defb 'SWA','P'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4007,10 +3450,7 @@
 	defb 002h
 RL_BXOR:
 	defw RL_SWAP
-	defb 042h
-	defb 058h
-	defb 04fh
-	defb 0d2h
+	defb 'BXO','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4018,9 +3458,7 @@
 	defb 003h
 RL_BOR:
 	defw RL_BXOR
-	defb 042h
-	defb 04fh
-	defb 0d2h
+	defb 'BO','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4028,10 +3466,7 @@
 	defb 003h
 RL_BAND:
 	defw RL_BOR
-	defb 042h
-	defb 041h
-	defb 04eh
-	defb 0c4h
+	defb 'BAN','D'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4039,53 +3474,37 @@
 	defb 003h
 RL_EXP:
 	defw RL_BAND
-	defb 045h
-	defb 058h
-	defb 0d0h
+	defb 'EX','P'+0x80
 	defb 00bh
 	defw Exp__
 RL_LN:
 	defw RL_EXP
-	defb 04ch
-	defb 0ceh
+	defb 'L','N'+0x80
 	defb 00bh
 	defw Ln__
 RL_ARCTAN:
 	defw RL_LN
-	defb 041h
-	defb 052h
-	defb 043h
-	defb 054h
-	defb 041h
-	defb 0ceh
+	defb 'ARCTA','N'+0x80
 	defb 00bh
 	defw Arctan__
 RL_TAN:
 	defw RL_ARCTAN
-	defb 054h
-	defb 041h
-	defb 0ceh
+	defb 'TA','N'+0x80
 	defb 00bh
 	defw Tan__
 RL_COS:
 	defw RL_TAN
-	defb 043h
-	defb 04fh
-	defb 0d3h
+	defb 'CO','S'+0x80
 	defb 00bh
 	defw Cos__
 RL_SIN:
 	defw RL_COS
-	defb 053h
-	defb 049h
-	defb 0ceh
+	defb 'SI','N'+0x80
 	defb 00bh
 	defw Sin__
 RL_INP:
 	defw RL_SIN
-	defb 049h
-	defb 04eh
-	defb 0d0h
+	defb 'IN','P'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -4093,53 +3512,32 @@
 	defb 002h
 RL_OUT:
 	defw RL_INP
-	defb 04fh
-	defb 055h
-	defb 0d4h
+	defb 'OU','T'+0x80
 	defb 006h
 	defw l5290h
 RL_SIZE:
 	defw RL_OUT
-	defb 053h
-	defb 049h
-	defb 05ah
-	defb 0c5h
+	defb 'SIZ','E'+0x80
 	defb 007h
 	defw l526bh
 RL_ADDR:
 	defw RL_SIZE
-	defb 041h
-	defb 044h
-	defb 044h
-	defb 0d2h
+	defb 'ADD','R'+0x80
 	defb 007h
 	defw l527fh
 RL_INLINE:
 	defw RL_ADDR
-	defb 049h
-	defb 04eh
-	defb 04ch
-	defb 049h
-	defb 04eh
-	defb 0c5h
+	defb 'INLIN','E'+0x80
 	defb 006h
 	defw l438ah
 RL_ENTIER:
 	defw RL_INLINE
-	defb 045h
-	defb 04eh
-	defb 054h
-	defb 049h
-	defb 045h
-	defb 0d2h
+	defb 'ENTIE','R'+0x80
 	defb 00ch
 	defw Entier__
 RL_USER:
 	defw RL_ENTIER
-	defb 055h
-	defb 053h
-	defb 045h
-	defb 0d2h
+	defb 'USE','R'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4147,12 +3545,7 @@
 	defb 002h
 RL_RANDOM:
 	defw RL_USER
-	defb 052h
-	defb 041h
-	defb 04eh
-	defb 044h
-	defb 04fh
-	defb 0cdh
+	defb 'RANDO','M'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4160,16 +3553,7 @@
 	defb 001h
 RL_KEYPRESSED:
 	defw RL_RANDOM
-	defb 04bh
-	defb 045h
-	defb 059h
-	defb 050h
-	defb 052h
-	defb 045h
-	defb 053h
-	defb 053h
-	defb 045h
-	defb 0c4h
+	defb 'KEYPRESSE','D'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4177,10 +3561,7 @@
 	defb 001h
 RL_HALT:
 	defw RL_KEYPRESSED
-	defb 048h
-	defb 041h
-	defb 04ch
-	defb 0d4h
+	defb 'HAL','T'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4188,10 +3569,7 @@
 	defb 001h
 RL_EOLN:
 	defw RL_HALT
-	defb 045h
-	defb 04fh
-	defb 04ch
-	defb 0ceh
+	defb 'EOL','N'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4199,10 +3577,7 @@
 	defb 001h
 RL_PAGE:
 	defw RL_EOLN
-	defb 050h
-	defb 041h
-	defb 047h
-	defb 0c5h
+	defb 'PAG','E'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4210,127 +3585,79 @@
 	defb 001h
 RL_SQRT:
 	defw RL_PAGE
-	defb 053h
-	defb 051h
-	defb 052h
-	defb 0d4h
+	defb 'SQR','T'+0x80
 	defb 00bh
 	defw Sqrt__
 RL_ROUND:
 	defw RL_SQRT
-	defb 052h
-	defb 04fh
-	defb 055h
-	defb 04eh
-	defb 0c4h
+	defb 'ROUN','D'+0x80
 	defb 00ch
 	defw Round__
 RL_TRUNC:
 	defw RL_ROUND
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 04eh
-	defb 0c3h
+	defb 'TRUN','C'+0x80
 	defb 00ch
 	defw Trunc__
 RL_MAXINT:
 	defw RL_TRUNC
-	defb 04dh
-	defb 041h
-	defb 058h
-	defb 049h
-	defb 04eh
-	defb 0d4h
+	defb 'MAXIN','T'+0x80
 	defb 001h
 	defb 001h
 	defb 000h
 	defw 07fffh
 RL_SUCC:
 	defw RL_MAXINT
-	defb 053h
-	defb 055h
-	defb 043h
-	defb 0c3h
+	defb 'SUC','C'+0x80
 	defb 007h
 	defw l436ch
 RL_PRED:
 	defw RL_SUCC
-	defb 050h
-	defb 052h
-	defb 045h
-	defb 0c4h
+	defb 'PRE','D'+0x80
 	defb 007h
 	defw l435bh
 RL_ORD:
 	defw RL_PRED
-	defb 04fh
-	defb 052h
-	defb 0c4h
+	defb 'OR','D'+0x80
 	defb 007h
 	defw l434ch
 RL_PEEK:
 	defw RL_ORD
-	defb 050h
-	defb 045h
-	defb 045h
-	defb 0cbh
+	defb 'PEE','K'+0x80
 	defb 007h
 	defw l4536h
 RL_POKE:
 	defw RL_PEEK
-	defb 050h
-	defb 04fh
-	defb 04bh
-	defb 0c5h
+	defb 'POK','E'+0x80
 	defb 006h
 	defw l3844h
 RL_RELEASE:
 	defw RL_POKE
-	defb 052h
-	defb 045h
-	defb 04ch
-	defb 045h
-	defb 041h
-	defb 053h
-	defb 0c5h
+	defb 'RELEAS','E'+0x80
 	defb 006h
 	defw l507bh
 RL_MARK:
 	defw RL_RELEASE
-	defb 04dh
-	defb 041h
-	defb 052h
-	defb 0cbh
+	defb 'MAR','K'+0x80
 	defb 006h
 	defw l5076h
 RL_NEW:
 	defw RL_MARK
-	defb 04eh
-	defb 045h
-	defb 0d7h
+	defb 'NE','W'+0x80
 	defb 006h
 	defw l509eh
 RL_TOUT:
 	defw RL_NEW
-	defb 054h
-	defb 04fh
-	defb 055h
-	defb 0d4h
+	defb 'TOU','T'+0x80
 	defb 006h
 	defw l50fah
 RL_TIN:
 	defw RL_TOUT
-	defb 054h
-	defb 049h
-	defb 0ceh
+	defb 'TI','N'+0x80
 	defb 006h
 	defw l50eeh
 RL_CHR:
 	defw RL_TIN
-	defb 043h
-	defb 048h
-	defb 0d2h
+	defb 'CH','R'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -4338,9 +3665,7 @@
 	defb 002h
 RL_ODD:
 	defw RL_CHR
-	defb 04fh
-	defb 044h
-	defb 0c4h
+	defb 'OD','D'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4348,27 +3673,19 @@
 	defb 002h
 RL_ABS:
 	defw RL_ODD
-	defb 041h
-	defb 042h
-	defb 0d3h
+	defb 'AB','S'+0x80
 	defb 00dh
 	defw l52e3h
 	defw l52eeh
 RL_SQR:
 	defw RL_ABS
-	defb 053h
-	defb 051h
-	defb 0d2h
+	defb 'SQ','R'+0x80
 	defb 00dh
 	defw l52dch
 	defw l52e8h
 RL_FALSE:
 	defw RL_SQR
-	defb 046h
-	defb 041h
-	defb 04ch
-	defb 053h
-	defb 0c5h
+	defb 'FALS','E'+0x80
 	defb 001h
 	defb 004h
 	defb 000h
@@ -4376,10 +3693,7 @@
 	defb 001h
 RL_TRUE:
 	defw RL_FALSE
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 0c5h
+	defb 'TRU','E'+0x80
 	defb 001h
 	defb 004h
 	defb 000h
@@ -4387,13 +3701,7 @@
 	defb 001h
 RL_BOOLEAN:
 	defw RL_TRUE
-	defb 042h
-	defb 04fh
-	defb 04fh
-	defb 04ch
-	defb 045h
-	defb 041h
-	defb 0ceh
+	defb 'BOOLEA','N'+0x80
 	defb 003h
 	defb 004h
 	defb 000h
@@ -4405,10 +3713,7 @@
 	defb 000h
 RL_CHAR:
 	defw RL_BOOLEAN
-	defb 043h
-	defb 048h
-	defb 041h
-	defb 0d2h
+	defb 'CHA','R'+0x80
 	defb 003h
 	defb 003h
 	defb 000h
@@ -4420,10 +3725,7 @@
 	defb 000h
 RL_REAL:
 	defw RL_CHAR
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 0cch
+	defb 'REA','L'+0x80
 	defb 003h
 	defb 002h
 	defb 000h
@@ -4435,13 +3737,7 @@
 	defb 000h
 RL_INTEGER:
 	defw RL_REAL
-	defb 049h
-	defb 04eh
-	defb 054h
-	defb 045h
-	defb 047h
-	defb 045h
-	defb 0d2h
+	defb 'INTEGE','R'+0x80
 	defb 003h
 	defb 001h
 	defb 000h
@@ -4453,2126 +3749,29 @@
 	defb 000h
 RL_READLN:
 	defw RL_INTEGER
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 044h
-	defb 04ch
-	defb 0ceh
+	defb 'READL','N'+0x80
 	defb 006h
 	defw l3ee5h
 RL_READ:
 	defw RL_READLN
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 0c4h
+	defb 'REA','D'+0x80
 	defb 006h
 	defw l3e95h
 RL_WRITELN:
 	defw RL_READ
-	defb 057h
-	defb 052h
-	defb 049h
-	defb 054h
-	defb 045h
-	defb 04ch
-	defb 0ceh
+	defb 'WRITEL','N'+0x80
 	defb 006h
 	defw CoWRITELN
 RL_WRITE:
 	defw RL_WRITELN
-	defb 057h
-	defb 052h
-	defb 049h
-	defb 054h
-	defb 0c5h
+	defb 'WRIT','E'+0x80
 	defb 006h
 	defw CoWRITE
 
 ; BLOCK 'Symtab_open' (start 0x1b35 end 0x2355)
 Symtab_open_start:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l1f40h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 2080
+
 START_PASCAL:
 	call GetRAMEnd
 l2358h:
@@ -7730,44 +4929,44 @@
 	xor a	
 	jr SetHLFirstSrcLine
 cmdTab:
-	defb 042h
+	defb 'B'
 	defw JEndPascal
-	defb 043h
+	defb 'C'
 	defw DoCmdC
-	defb 044h
+	defb 'D'
 	defw DoCmdD
-	defb 045h
+	defb 'E'
 	defw DoCmdE
-	defb 046h
+	defb 'F'
 	defw DoCmdF
-	defb 047h
+	defb 'G'
 	defw DoCmdG
-	defb 049h
+	defb 'I'
 	defw DoCmdI
-	defb 04ch
+	defb 'L'
 	defw DoCmdL
-	defb 04bh
+	defb 'K'
 	defw DoCmdK
-	defb 04dh
+	defb 'M'
 	defw DoCmdM
-	defb 04eh
+	defb 'N'
 	defw DoCmdN
-	defb 04fh
+	defb 'O'
 	defw DoCmdO
-	defb 050h
+	defb 'P'
 	defw DoCmdP
-	defb 052h
+	defb 'R'
 cmdTabRunAddr:
 	defw PasPrgStart
-	defb 053h
+	defb 'S'
 	defw DoCmdS
-	defb 054h
+	defb 'T'
 	defw DoCmdT
-	defb 056h
+	defb 'V'
 	defw DoCmdV
-	defb 058h
+	defb 'X'
 	defw DoCmdX
-	defb 05ah
+	defb 'Z'
 	defw DoCmdZ
 EdCmdTab:
 	defb 00bh
@@ -7776,82 +4975,36 @@
 	defw sub_2619h
 	defb 008h
 	defw sub_2668h
-	defb 043h
+	defb 'C'
 	defw l2672h
 	defb 00dh
 	defw l28e8h
-	defb 046h
+	defb 'F'
 	defw l26e8h
-	defb 049h
+	defb 'I'
 	defw l267ch
-	defb 04bh
+	defb 'K'
 	defw l2636h
-	defb 04ch
+	defb 'L'
 	defw sub_29d4h
-	defb 051h
+	defb 'Q'
 	defw l29f5h
-	defb 053h
+	defb 'S'
 	defw l2692h
-	defb 052h
+	defb 'R'
 	defw l2561h
-	defb 058h
+	defb 'X'
 	defw l2677h
-	defb 05ah
+	defb 'Z'
 	defw l2681h
 tPardon:
-	defb 050h
-	defb 061h
-	defb 072h
-	defb 064h
-	defb 06fh
-	defb 06eh
-	defb 03fh
-	defb 00dh
-	defb 000h
+	defb 'Pardon?',00dh,000h
 param3:
 	defb 00dh
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 20
 param4:
 	defb 00dh
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 20
 l2b33h:
 	nop	
 lineNumStart__:
@@ -7893,18 +5046,9 @@
 listPageSize:
 	nop	
 GetSrcChr:
-	jp l5491h
+	jp 05491h
 curIdentifier:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 10
 SP_safe:
 	defb 0f0h
 	defb 07fh
@@ -8117,81 +5261,10 @@
 banner_nl:
 	defb 00dh
 banner:
-	defb 00ch
-	defb 012h
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 020h
-	defb 04bh
-	defb 043h
-	defb 02dh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 043h
-	defb 041h
-	defb 04ch
-	defb 020h
-	defb 056h
-	defb 035h
-	defb 02eh
-	defb 031h
-	defb 020h
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 00dh
-	defb 00dh
-	defb 042h
-	defb 045h
-	defb 041h
-	defb 052h
-	defb 042h
-	defb 02eh
-	defb 020h
-	defb 056h
-	defb 04fh
-	defb 04eh
-	defb 020h
-	defb 020h
-	defb 02bh
-	defb 02bh
-	defb 02bh
-	defb 020h
-	defb 041h
-	defb 04dh
-	defb 039h
-	defb 030h
-	defb 020h
-	defb 02bh
-	defb 02bh
-	defb 02bh
-	defb 00dh
-	defb 00dh
-	defb 020h
-	defb 020h
-	defb 020h
-	defb 028h
-	defb 056h
-	defb 045h
-	defb 052h
-	defb 053h
-	defb 049h
-	defb 04fh
-	defb 04eh
-	defb 020h
-	defb 04bh
-	defb 043h
-	defb 038h
-	defb 035h
-	defb 02fh
-	defb 034h
-	defb 029h
-	defb 00dh
-	defb 000h
+ 	defb 00ch,012h
+ 	defb '**** KC-PASCAL V5.1 ****',00dh,00dh
+ 	defb 'BEARB. VON  +++ AM90 +++',00dh,00dh
+ 	defb '   (VERSION KC85/4)',00dh,000h
 CoVarInitData:
 	defb 020h
 	defw RL_PACKED
@@ -8537,17 +5610,9 @@
 	call PrNL
 	jp PasPrgStart
 TOk:
-	defb 04fh
-	defb 06bh
-	defb 03fh
-	defb 000h
+	defb 'Ok?',000h
 TLauf:
-	defb 04ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 03fh
-	defb 000h
+	defb 'Lauf?',000h
 CSq_JReset:
 	defb 003h
 	jp Reset
@@ -8877,345 +5942,171 @@
 
 ; BLOCK 'compOptTab' (start 0x3165 end 0x3171)
 compOptTab_start:
-	defb 04ch
+	defb 'L'
 	defb 001h
-	defb 04fh
+	defb 'O'
 	defb 002h
-	defb 043h
+	defb 'C'
 	defb 004h
-	defb 053h
+	defb 'S'
 	defb 008h
-	defb 049h
+	defb 'I'
 	defb 010h
-	defb 041h
+	defb 'A'
 	defb 020h
 RL_PACKED:
 	defw 00000h
-	defb 050h
-	defb 041h
-	defb 043h
-	defb 04bh
-	defb 045h
-	defb 0c4h
+	defb 'PACKE','D'+080h
 	defb 023h
 RL_NIL:
 	defw RL_PACKED
-	defb 04eh
-	defb 049h
-	defb 0cch
+	defb 'NI','L'+080h
 	defb 022h
 RL_FORWARD:
 	defw RL_NIL
-	defb 046h
-	defb 04fh
-	defb 052h
-	defb 057h
-	defb 041h
-	defb 052h
-	defb 0c4h
+	defb 'FORWAR','D'+080h
 	defb 01dh
 RL_PROGRAM:
 	defw RL_FORWARD
-	defb 050h
-	defb 052h
-	defb 04fh
-	defb 047h
-	defb 052h
-	defb 041h
-	defb 0cdh
+	defb 'PROGRA','M'+080h
 	defb 001h
 RL_IN:
 	defw RL_PROGRAM
-	defb 049h
-	defb 0ceh
+	defb 'I','N'+080h
 	defb 020h
 RL_OR:
 	defw RL_IN
-	defb 04fh
-	defb 0d2h
+	defb 'O','R'+080h
 	defb 007h
-RL_ON:
+RL_OF:
 	defw RL_OR
-	defb 04fh
-	defb 0c6h
+	defb 'O','F'+080h
 	defb 00bh
 RL_TO:
-	defw RL_ON
-	defb 054h
-	defb 0cfh
+	defw RL_OF
+	defb 'T','O'+080h
 	defb 00ch
 RL_DO:
 	defw RL_TO
-	defb 044h
-	defb 0cfh
+	defb 'D','O'+080h
 	defb 011h
 RL_IF:
 	defw RL_DO
-	defb 049h
-	defb 0c6h
+	defb 'I','F'+080h
 	defb 017h
 RL_SET:
 	defw RL_IF
-	defb 053h
-	defb 045h
-	defb 0d4h
+	defb 'SE','T'+080h
 	defb 01bh
 RL_NOT:
 	defw RL_SET
-	defb 04eh
-	defb 04fh
-	defb 0d4h
+	defb 'NO','T'+080h
 	defb 006h
 RL_MOD:
 	defw RL_NOT
-	defb 04dh
-	defb 04fh
-	defb 0c4h
+	defb 'MO','D'+080h
 	defb 009h
 RL_DIV:
 	defw RL_MOD
-	defb 044h
-	defb 049h
-	defb 0d6h
+	defb 'DI','V'+080h
 	defb 002h
 RL_VAR:
 	defw RL_DIV
-	defb 056h
-	defb 041h
-	defb 0d2h
+	defb 'VA','R'+080h
 	defb 00ah
 RL_AND:
 	defw RL_VAR
-	defb 041h
-	defb 04eh
-	defb 0c4h
+	defb 'AN','D'+080h
 	defb 008h
 RL_FOR:
 	defw RL_AND
-	defb 046h
-	defb 04fh
-	defb 0d2h
+	defb 'FO','R'+080h
 	defb 016h
 RL_END:
 	defw RL_FOR
-	defb 045h
-	defb 04eh
-	defb 0c4h
+	defb 'EN','D'+080h
 	defb 010h
 RL_GOTO:
 	defw RL_END
-	defb 047h
-	defb 04fh
-	defb 054h
-	defb 0cfh
+	defb 'GOT','O'+080h
 	defb 01ah
 RL_WITH:
 	defw RL_GOTO
-	defb 057h
-	defb 049h
-	defb 054h
-	defb 0c8h
+	defb 'WIT','H'+080h
 	defb 019h
 RL_TYPE:
 	defw RL_WITH
-	defb 054h
-	defb 059h
-	defb 050h
-	defb 0c5h
+	defb 'TYP','E'+080h
 	defb 01fh
 RL_CASE:
 	defw RL_TYPE
-	defb 043h
-	defb 041h
-	defb 053h
-	defb 0c5h
+	defb 'CAS','E'+080h
 	defb 014h
 RL_ELSE:
 	defw RL_CASE
-	defb 045h
-	defb 04ch
-	defb 053h
-	defb 0c5h
+	defb 'ELS','E'+080h
 	defb 012h
 RL_THEN:
 	defw RL_ELSE
-	defb 054h
-	defb 048h
-	defb 045h
-	defb 0ceh
+	defb 'THE','N'+080h
 	defb 00eh
 RL_LABEL:
 	defw RL_THEN
-	defb 04ch
-	defb 041h
-	defb 042h
-	defb 045h
-	defb 0cch
+	defb 'LABE','L'+080h
 	defb 021h
 RL_CONST:
 	defw RL_LABEL
-	defb 043h
-	defb 04fh
-	defb 04eh
-	defb 053h
-	defb 0d4h
+	defb 'CONS','T'+080h
 	defb 003h
 RL_ARRAY:
 	defw RL_CONST
-	defb 041h
-	defb 052h
-	defb 052h
-	defb 041h
-	defb 0d9h
+	defb 'ARRA','Y'+080h
 	defb 01ch
 RL_UNTIL:
 	defw RL_ARRAY
-	defb 055h
-	defb 04eh
-	defb 054h
-	defb 049h
-	defb 0cch
+	defb 'UNTI','L'+080h
 	defb 00fh
 RL_WHILE:
 	defw RL_UNTIL
-	defb 057h
-	defb 048h
-	defb 049h
-	defb 04ch
-	defb 0c5h
+	defb 'WHIL','E'+080h
 	defb 015h
 RL_BEGIN:
 	defw RL_WHILE
-	defb 042h
-	defb 045h
-	defb 047h
-	defb 049h
-	defb 0ceh
+	defb 'BEGI','N'+080h
 	defb 018h
 RL_RECORD:
-	defw RL_BEGIN
-	defb 052h
-	defb 045h
-	defb 043h
-	defb 04fh
-	defb 052h
-	defb 0c4h
-	defb 01eh
+ 	defw RL_BEGIN
+	defb 'RECOR','D'+080h
+ 	defb 01eh
 RL_DOWNTO:
 	defw RL_RECORD
-	defb 044h
-	defb 04fh
-	defb 057h
-	defb 04eh
-	defb 054h
-	defb 0cfh
+	defb 'DOWNT','O'+080h
 	defb 00dh
 RL_REPEAT:
-	defw RL_DOWNTO
-	defb 052h
-	defb 045h
-	defb 050h
-	defb 045h
-	defb 041h
-	defb 0d4h
-	defb 013h
+ 	defw RL_DOWNTO
+	defb 'REPEA','T'+080h
+ 	defb 013h
 RL_FUNCTION:
-	defw RL_REPEAT
-	defb 046h
-	defb 055h
-	defb 04eh
-	defb 043h
-	defb 054h
-	defb 049h
-	defb 04fh
-	defb 0ceh
-	defb 005h
+ 	defw RL_REPEAT
+	defb 'FUNCTIO','N'+080h
+ 	defb 005h
 ResWordsEntry2:
 RL_PROCEDURE:
-	defw RL_FUNCTION
-	defb 050h
-	defb 052h
-	defb 04fh
-	defb 043h
-	defb 045h
-	defb 044h
-	defb 055h
-	defb 052h
-	defb 0c5h
-	defb 004h
+ 	defw RL_FUNCTION
+	defb 'PROCEDUR','E'+080h
+ 	defb 004h
 ResWordsEntry:
-	defw ResWordsEntry2
+ 	defw ResWordsEntry2
 tEadr:
-	defb 045h
-	defb 06eh
-	defb 064h
-	defb 061h
-	defb 064h
-	defb 072h
-	defb 065h
-	defb 073h
-	defb 073h
-	defb 065h
-	defb 03ah
-	defb 020h
-	defb 000h
+	defb 'Endadresse: ',000h
 tFeh1:
-	defb 046h
-	defb 065h
-	defb 068h
-	defb 06ch
-	defb 065h
-	defb 072h
-	defb 03ah
-	defb 000h
+	defb 'Fehler:',000h
 tFeh2:
-	defb 02ah
-	defb 046h
-	defb 045h
-	defb 048h
-	defb 04ch
-	defb 045h
-	defb 052h
-	defb 02ah
-	defb 000h
+	defb '*FEHLER*',000h
 tKeTx:
-	defb 00dh
-	defb 04bh
-	defb 065h
-	defb 069h
-	defb 06eh
-	defb 020h
-	defb 054h
-	defb 065h
-	defb 078h
-	defb 074h
-	defb 020h
-	defb 06dh
-	defb 065h
-	defb 068h
-	defb 072h
-	defb 021h
-	defb 000h
+	defb 00dh,'Kein Text mehr!',000h
 tTabU:
-	defb 054h
-	defb 061h
-	defb 062h
-	defb 065h
-	defb 06ch
-	defb 06ch
-	defb 065h
-	defb 06eh
-	defb 075h
-	defb 065h
-	defb 062h
-	defb 065h
-	defb 072h
-	defb 06ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 021h
-	defb 000h
+	defb 'Tabellenueberlauf!',000h
 ChkTypeBool__:
 	ld de,00004h
 	jr ChkType__
@@ -9838,7 +6729,6 @@
 l35f0h:
 	defb 008h
 	dec b	
-l35f2h:
 	djnz l35ech
 	inc a	
 l35f5h:
@@ -10098,7 +6988,7 @@
 	add a,a	
 	ld l,a	
 	ld h,000h
-	ld de,0371eh
+	ld de,xxTab4__-00026h
 	add hl,de	
 	ld e,(hl)	
 	inc hl	
@@ -10110,7 +7000,7 @@
 	ret	
 sub_376dh:
 	ex af,af'	
-	ld de,l1110h
+	ld de,01110h
 	call ChkLexem_GetLex
 	call WCaBreak_IfCoCo_C
 l3777h:
@@ -10914,7 +7804,7 @@
 	call WCaBreak_IfCoCo_C
 	call sub_3710h
 	push hl	
-	ld de,l1110h
+	ld de,01110h
 	call ChkLexem_GetLex
 	call l3777h
 	call WrJump
@@ -10927,7 +7817,7 @@
 l3c8ah:
 	call sub_3710h
 	push hl	
-	ld de,l0e0fh
+	ld de,00e0fh
 	call ChkLexem_GetLex
 	call l3777h
 	cp 012h
@@ -11016,7 +7906,7 @@
 	push hl	
 	cp 0ach
 	jr z,l3d3ch
-	ld de,l1110h
+	ld de,01110h
 	call ChkLexem_GetLex
 	call l3777h
 l3d32h:
@@ -11379,7 +8269,7 @@
 	ld hl,CSq_l4184h
 	call WCodeOverLastByte
 l3f78h:
-	ld bc,l360ah
+	ld bc,TabCSq-000beh
 WCode_VarIdxTOS:
 	pop hl	
 	call GetCSqAddrByIdx
@@ -11387,16 +8277,16 @@
 l3f82h:
 	pop de	
 	call ChkType__
-	ld bc,l35f2h
+	ld bc,TabCSq-000d6h
 	jr WCode_VarIdxTOS
 l3f8bh:
 	ld a,e	
 	pop de	
 	call ChkType__
 	bit 4,(ix+000h)
-	ld bc,035dah
+	ld bc,TabCSq-000eeh
 	jr z,WCode_VarIdxTOS
-	ld bc,035e6h
+	ld bc,TabCSq-000e2h
 	jr WCode_VarIdxTOS
 l3f9eh:
 	ld a,e	
@@ -11411,7 +8301,7 @@
 	call sub_343dh
 	ld hl,CSq_l35e8h
 	call WCode
-	ld bc,035feh
+	ld bc,TabCSq-000cah
 	jr WCode_VarIdxTOS
 l3fbbh:
 	pop hl	
@@ -11422,7 +8312,7 @@
 	cp 0f4h
 	ld a,e	
 	jr z,l3fd7h
-	ld bc,l3616h
+	ld bc,TabCSq-000b2h
 	call GetCSqAddrByIdx
 	jp CodeLdBC_0n_from_0x2b8b
 l3fd0h:
@@ -11442,7 +8332,7 @@
 	push hl	
 	ld a,l	
 	cp 0f2h
-	ld bc,035dah
+	ld bc,TabCSq-000eeh
 	ld a,e	
 	jr c,WCode_VarIdxTOS
 	ld e,040h
@@ -11718,7 +8608,6 @@
 l419eh:
 	call l381ah
 	djnz l419eh
-l41a3h:
 	ret	
 l41a4h:
 	inc a	
@@ -11941,7 +8830,6 @@
 	ld e,(hl)	
 	inc hl	
 	ld d,(hl)	
-l4305h:
 	push de	
 	inc hl	
 	ld b,(hl)	
@@ -13124,7 +10012,7 @@
 l4aafh:
 	ld a,c	
 ChkBegin:
-	ld de,l1819h
+	ld de,01819h
 	call ChkLexem_GetLex
 	jp l3cd4h
 WCorrAddr:
@@ -14003,7 +10891,6 @@
 	jp l4e61h
 l4fffh:
 	push hl	
-l5000h:
 	ld d,(ix+001h)
 	push de	
 	ld c,001h
@@ -14165,7 +11052,7 @@
 	jr l50f4h
 sub_5105h:
 	call NextChkOpBra_GetLex
-	ld bc,l0208h
+	ld bc,00208h
 	call sub_3f29h
 sub_510eh:
 	call ChkComma_GetLex
@@ -14493,19 +11380,8 @@
 	push hl	
 	push de	
 StartPASSrc:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 13
+
 PasEx:
 	ld hl,00000h
 	ld b,080h
@@ -14513,7 +11389,6 @@
 	ld a,(hl)	
 	cp 07fh
 	inc hl	
-l5309h:
 	jr z,PXKillProlog
 	djnz PXSrchProlog
 	jr PXCpChrMap
@@ -14522,7 +11397,7 @@
 PXCpChrMap:
 	ld hl,(CCTL0)
 	ld de,0ba00h
-	ld bc,PasPrgMenuHdr
+	ld bc,00200h
 	ldir
 	ld hl,PXUSASC
 	ld de,0bbd8h
@@ -14530,7 +11405,7 @@
 	ldir
 	ld hl,PXBASCI
 	ld de,PXSASCI
-	ld bc,0000fh
+	ld bc,PXTAPE-PXSASCI
 	ldir
 	ld a,07fh
 	ld (PXSASCI),a
@@ -14539,7 +11414,7 @@
 	in a,(c)
 	cp 0a7h
 	jp nz,PXASCI
-	ld bc,000dfh
+	ld bc,SUBNEU-PXTAPE
 	ldir
 	ld a,07fh
 	ld (PXTAPE),a
@@ -14567,66 +11442,36 @@
 	defw ISRI
 	defw CSRI
 PXUSASC:
-	defb 07ch
-	defb 060h
-	defb 060h
-	defb 060h
-	defb 060h
-	defb 060h
-	defb 07ch
-	defb 000h
-	defb 0c0h
-	defb 060h
-	defb 030h
-	defb 018h
-	defb 00ch
-	defb 006h
-	defb 002h
-	defb 000h
-	defb 07ch
-	defb 00ch
-	defb 00ch
-	defb 00ch
-	defb 00ch
-	defb 00ch
-	defb 07ch
-	defb 000h
+	defb 07ch,060h,060h,060h,060h,060h,07ch,000h
+	defb 0c0h,060h,030h,018h,00ch,006h,002h,000h
+	defb 07ch,00ch,00ch,00ch,00ch,00ch,07ch,000h
 PXBASCI:
+
+	org  0bc00h
+
+PXSASCI:
 	defw 00000h
-	defb 041h
-	defb 053h
-	defb 043h
-	defb 049h
-	defb 049h
+	defb 'ASCII'
 	defb 001h
+PXASCI:
 	ld hl,0ba00h
 	ld (CCTL0),hl
 	ret	
+PXTAPE:
 	defw 00000h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 054h
-	defb 041h
-	defb 050h
-	defb 045h
+	defb 'PASTAPE'
 	defb 001h
 	ld hl,(SUBALT)
 PXSetSUTAB:
 	ld (SUTAB),hl
 	ret	
+PXDISK:
 	defw 00000h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 044h
-	defb 049h
-	defb 053h
-	defb 04bh
+	defb 'PASDISK'
 	defb 001h
 	ld hl,SUBNEU
 	jr PXSetSUTAB
-ISRO_s:
+ISRO:
 	ld (ix+002h),000h
 	ld l,(ix+005h)
 	ld h,(ix+006h)
@@ -14639,7 +11484,7 @@
 	dec e	
 	jr nz,PXSendName
 	ld d,00bh
-PXNxtBlock_s:
+PXNxtBlock:
 	inc (ix+002h)
 	ld h,(ix+006h)
 	ld l,(ix+005h)
@@ -14651,7 +11496,7 @@
 	inc b	
 	dec e	
 	jr nz,PXSendBlock
-PXSendCtl_s:
+PXSendCtl:
 	ld bc,080f3h
 	out (c),d
 PXWaitForReady:
@@ -14674,7 +11519,7 @@
 	defb 019h
 	scf	
 	ret	
-MBO_s:
+MBO:
 	ld d,003h
 	call PXNxtBlock
 	ret c	
@@ -14683,29 +11528,26 @@
 	ret nc	
 	call PV1
 	defb 023h
-	defb 008h
-	defb 008h
-	defb 008h
-	defb 000h
+	defb 008h,008h,008h,000h
 	and a	
 	ret	
-CSRO_s:
+CSRO:
 	call MBO
 	ret c	
 	ld d,043h
-	jr PXSendCtl_s
-ISRI_s:
+	jr PXSendCtl
+ISRI:
 	ld (ix+002h),000h
 	ld hl,fileName
 	ld bc,083f3h
-	ld de,ErrIdxHigh
+	ld de,0090bh
 PXSendName2:
 	outi
 	inc b	
 	inc b	
 	dec e	
 	jr nz,PXSendName2
-PXRecvBlock_s:
+PXRecvBlock:
 	call PXSendCtl
 	ret c	
 	push hl	
@@ -14724,7 +11566,7 @@
 	pop af	
 	pop hl	
 	ret	
-MBI_s:
+MBI:
 	push de	
 	ld d,001h
 	call PXRecvBlock
@@ -14735,131 +11577,28 @@
 	ret nc	
 	call PV1
 	defb 023h
-	defb 008h
-	defb 008h
-	defb 008h
-	defb 008h
-	defb 000h
+	defb 008h,008h,008h,008h,000h
 	and a	
 	ret	
-CSRI_s:
+CSRI:
 	call PV1
 	defb 02ch
 	ret	
-SUBNEU_s:
+SUBNEU:
 
 ; BLOCK 'Rest' (start 0x5490 end 0x5500)
 Rest_first:
-	defb 00dh
-l5491h:
-	defb 00ah
-	defb 009h
-	defb 043h
-	defb 050h
-	defb 009h
-	defb 037h
-	defb 046h
-	defb 048h
-	defb 009h
-	defb 03bh
-	defb 054h
-	defb 065h
-	defb 073h
-	defb 074h
-	defb 020h
-	defb 050h
-	defb 072h
-	defb 06fh
-	defb 06ch
-	defb 06fh
-	defb 067h
-	defb 03fh
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 049h
-	defb 04eh
-	defb 043h
-	defb 009h
-	defb 048h
-	defb 04ch
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 04ah
-	defb 052h
-	defb 009h
-	defb 05ah
-	defb 02ch
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 031h
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 044h
-	defb 04ah
-	defb 04eh
-	defb 05ah
-	defb 009h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 032h
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 04ah
-	defb 052h
-	defb 009h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 033h
-	defb 00dh
-	defb 00ah
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 031h
-	defb 009h
-	defb 04ch
-	defb 044h
-	defb 009h
-	defb 04dh
-	defb 02ch
-	defb 030h
-	defb 009h
-	defb 03bh
-	defb 06ch
-	defb 07ch
-	defb 073h
-	defb 063h
-	defb 068h
-	defb 065h
-	defb 06eh
-	defb 00dh
-	defb 00ah
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 033h
-	defb 009h
-	defb 04ch
-	defb 044h
-	defb 009h
-	defb 048h
-	defb 04ch
-	defb 02ch
-	defb 028h
-	defb 043h
-	defb 043h
-	defb 054h
-	defb 04ch
-	defb 030h
+	defb 00dh,00ah,009h,043h,050h,009h,037h,046h ; ...CP.7F 
+	defb 048h,009h,03bh,054h,065h,073h,074h,020h ; H.;Test  
+	defb 050h,072h,06fh,06ch,06fh,067h,03fh,00dh ; Prolog?. 
+	defb 00ah,009h,049h,04eh,043h,009h,048h,04ch ; ..INC.HL 
+	defb 00dh,00ah,009h,04ah,052h,009h,05ah,02ch ; ...JR.Z, 
+	defb 050h,041h,053h,030h,031h,00dh,00ah,009h ; PAS01... 
+	defb 044h,04ah,04eh,05ah,009h,050h,041h,053h ; DJNZ.PAS 
+	defb 030h,032h,00dh,00ah,009h,04ah,052h,009h ; 02...JR. 
+	defb 050h,041h,053h,030h,033h,00dh,00ah,050h ; PAS03..P 
+	defb 041h,053h,030h,031h,009h,04ch,044h,009h ; AS01.LD. 
+	defb 04dh,02ch,030h,009h,03bh,06ch,07ch,073h ; M,0.;l|s 
+	defb 063h,068h,065h,06eh,00dh,00ah,050h,041h ; chen..PA 
+	defb 053h,030h,033h,009h,04ch,044h,009h,048h ; S03.LD.H 
+	defb 04ch,02ch,028h,043h,043h,054h,04ch,030h ; L,(CCTL0 
