--- p51.asm	2025-01-23 22:21:20.055212791 +0100
+++ pascal51.asm	2025-01-23 22:22:06.793905668 +0100
@@ -37,38 +37,16 @@
 l0200h:
 	defw 00000h
 PasPrgMenuName:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l0208h:
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h,000h,000h,000h,000h,000h
 	defb 000h
 	jp PasPrgStart
 MenuPRec:
 	defw 07f7fh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 052h
-	defb 045h
-	defb 043h
-	defb 000h
+	defb 'RASREC',000h
 	jp Recall
 MenuPEnt:
 	defw 07f7fh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 045h
-	defb 04eh
-	defb 054h
-	defb 052h
-	defb 059h
-	defb 000h
+	defb 'PASENTRY',000h
 	jp Entry
 DoNopRET:
 	nop	
@@ -719,18 +697,9 @@
 JCompileToRuntimeEnd:
 	jp SetBinStartToRuntimeEnd
 fileName:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h,000h,000h,000h,000h,000h
 fileExt:
-	defb 000h
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h
 	defb 000h
 l0704h:
 	defw 00000h
@@ -842,13 +811,9 @@
 	ld hl,fileExtCOM
 	jr SetFileExt
 fileExtCOM:
-	defb 043h
-	defb 04fh
-	defb 04dh
+	defb 'COM'
 fileExtPAS:
-	defb 050h
-	defb 041h
-	defb 053h
+	defb 'PAS'
 SaveSrcFile:
 	push ix
 	push iy
@@ -1048,21 +1013,12 @@
 
 ; BLOCK 'TFalse' (start 0x08d4 end 0x08da)
 TFalse_start:
-	defb 046h
-	defb 041h
-	defb 04ch
-	defb 053h
-	defb 045h
-	defb 000h
+	defb 'FALSE',000h
 tTrue:
 
 ; BLOCK 'TTrue' (start 0x08da end 0x08df)
 TTrue_start:
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 045h
-	defb 000h
+	defb 'TRUE',000h
 PrRAM:
 	ld de,T_RAM
 	jr PrErr2
@@ -1121,140 +1077,27 @@
 ; BLOCK 'TErrMsg' (start 0x0934 end 0x09b1)
 TErrMsg_start:
 	defb 00dh
-	defb 048h
-	defb 061h
-	defb 06ch
-	defb 074h
+	defb 'Halt'
 T_beiPC:
-	defb 020h
-	defb 062h
-	defb 065h
-	defb 069h
-	defb 020h
-	defb 050h
-	defb 043h
-	defb 03dh
-	defb 000h
+	defb ' bei PC=',000h
 TUeber:
-	defb 055h
-	defb 065h
-	defb 062h
-	defb 065h
-	defb 072h
-	defb 06ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 000h
+	defb 'Ueberlauf',000h
 T_RAM:
-	defb 03eh
-	defb 03eh
-	defb 020h
-	defb 052h
-	defb 041h
-	defb 04dh
-	defb 000h
+	defb '>> RAM',000h
 TDiv0:
-	defb 02fh
-	defb 020h
-	defb 064h
-	defb 075h
-	defb 072h
-	defb 063h
-	defb 068h
-	defb 020h
-	defb 04eh
-	defb 075h
-	defb 06ch
-	defb 06ch
-	defb 000h
+	defb '/ durch Null',000h
 TIdxLow:
-	defb 049h
-	defb 06eh
-	defb 064h
-	defb 065h
-	defb 078h
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 06eh
-	defb 069h
-	defb 065h
-	defb 064h
-	defb 072h
-	defb 069h
-	defb 067h
-	defb 000h
+	defb 'Index zu niedrig',000h
 TIdxHigh:
-	defb 049h
-	defb 06eh
-	defb 064h
-	defb 065h
-	defb 078h
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 068h
-	defb 06fh
-	defb 063h
-	defb 068h
-	defb 000h
+	defb 'Index zu hoch',000h
 TMathErr:
-	defb 06dh
-	defb 061h
-	defb 074h
-	defb 068h
-	defb 065h
-	defb 06dh
-	defb 061h
-	defb 074h
-	defb 069h
-	defb 073h
-	defb 063h
-	defb 068h
-	defb 065h
-	defb 072h
-	defb 020h
-	defb 046h
-	defb 065h
-	defb 068h
-	defb 06ch
-	defb 065h
-	defb 072h
-	defb 000h
+	defb 'mathematischer Fehler',000h
 TNumTooBig:
-	defb 05ah
-	defb 061h
-	defb 068h
-	defb 06ch
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 067h
-	defb 072h
-	defb 06fh
-	defb 07eh
-	defb 000h
+	defb 'Zahl zu gro',07eh,000h
 TEmpty:
 	defb 000h
 TNumExpected:
-	defb 05ah
-	defb 061h
-	defb 068h
-	defb 06ch
-	defb 020h
-	defb 065h
-	defb 072h
-	defb 077h
-	defb 061h
-	defb 072h
-	defb 074h
-	defb 065h
-	defb 074h
-	defb 000h
+	defb 'Zahl erwartet',000h
 SelListElem__:
 	ld d,ixh
 	ld e,ixl
@@ -2651,15 +2494,8 @@
 	defb 0e2h
 	defb 053h
 realStrPart:
-	defb 020h
-	defb 030h
-	defb 02eh
-	defb 000h
-	defb 045h
-	defb 02bh
-	defb 030h
-	defb 030h
-	defb 000h
+	defb ' 0.',000h
+        defb 'E+00',000h
 Sqrt__:
 	ld a,h	
 	or a	
@@ -2850,24 +2686,7 @@
 	ld de,tExpErw
 	jp PrErr2
 tExpErw:
-	defb 045h
-	defb 078h
-	defb 070h
-	defb 06fh
-	defb 06eh
-	defb 065h
-	defb 06eh
-	defb 074h
-	defb 020h
-	defb 065h
-	defb 072h
-	defb 077h
-	defb 061h
-	defb 072h
-	defb 074h
-	defb 065h
-	defb 074h
-	defb 000h
+	defb 'Exponent erwartet',000h
 Frac__:
 	ld a,h	
 	or a	
@@ -3573,47 +3392,9 @@
 	call RealMul__
 	jp l150eh
 l1761h:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 8
 l1769h:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 32
 retAddr__:
 	defw 00000h
 l178bh:
@@ -3648,175 +3429,11 @@
 iBufCurChrAddr:
 	defw 00000h
 inputBuf:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 85
 RuntimeEnd:
 	nop	
 lineBuf:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l1819h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 80
 lineBufEnd:
 	defb 00dh
 stack_adr:
@@ -3828,23 +3445,13 @@
 	defw START_PASCAL
 RL_SETSYS:
 	defw 00000h
-	defb 053h
-	defb 045h
-	defb 054h
-	defb 053h
-	defb 059h
-	defb 0d3h
+	defb 'SETSY','S'+0x80
 	defb 006h
 	defb 04ah
 	defb 051h
 RL_GETSYS:
 	defw RL_SETSYS
-	defb 047h
-	defb 045h
-	defb 054h
-	defb 053h
-	defb 059h
-	defb 0d3h
+	defb 'GETSY','S'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3853,32 +3460,19 @@
 	defb 002h
 RL_PLOT:
 	defw RL_GETSYS
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'PLO','T'+0x80
 	defb 006h
 	defb 074h
 	defb 051h
 RL_CLRPLOT:
 	defw RL_PLOT
-	defb 043h
-	defb 04ch
-	defb 052h
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'CLRPLO','T'+0x80
 	defb 006h
 	defb 0a0h
 	defb 051h
 RL_PTEST:
 	defw RL_CLRPLOT
-	defb 050h
-	defb 054h
-	defb 045h
-	defb 053h
-	defb 0d4h
+	defb 'PTES','T'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -3887,10 +3481,7 @@
 	defb 003h
 RL_GETC:
 	defw RL_PTEST
-	defb 047h
-	defb 045h
-	defb 054h
-	defb 0c3h
+	defb 'GET','C'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3899,52 +3490,31 @@
 	defb 003h
 RL_SETC:
 	defw RL_GETC
-	defb 053h
-	defb 045h
-	defb 054h
-	defb 0c3h
+	defb 'SET','C'+0x80
 	defb 006h
 	defb 060h
 	defb 051h
 RL_LINEPLOT:
 	defw RL_SETC
-	defb 04ch
-	defb 049h
-	defb 04eh
-	defb 045h
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'LINEPLO','T'+0x80
 	defb 006h
 	defb 0b4h
 	defb 051h
 RL_CIRCLE:
 	defw RL_LINEPLOT
-	defb 043h
-	defb 049h
-	defb 052h
-	defb 043h
-	defb 04ch
-	defb 0c5h
+	defb 'CIRCL','E'+0x80
 	defb 006h
 	defb 0d0h
 	defb 051h
 RL_GOTOXY:
 	defw RL_CIRCLE
-	defb 047h
-	defb 04fh
-	defb 054h
-	defb 04fh
-	defb 058h
-	defb 0d9h
+	defb 'GOTOX','Y'+0x80
 	defb 006h
 	defb 036h
 	defb 051h
 RL_PI:
 	defw RL_GOTOXY
-	defb 050h
-	defb 0c9h
+	defb 'P','I'+0x80
 	defb 001h
 	defb 002h
 	defb 000h
@@ -3954,21 +3524,12 @@
 	defb 064h
 RL_FRAC:
 	defw RL_PI
-	defb 046h
-	defb 052h
-	defb 041h
-	defb 0c3h
+	defb 'FRA','C'+0x80
 	defb 00bh
 	defw Frac__
 RL_READKBD:
 	defw RL_FRAC
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 044h
-	defb 04bh
-	defb 042h
-	defb 0c4h
+	defb 'READKB','D'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -3977,9 +3538,7 @@
 	defb 001h
 RL_SHR:
 	defw RL_READKBD
-	defb 053h
-	defb 048h
-	defb 0d2h
+	defb 'SH','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3988,9 +3547,7 @@
 	defb 003h
 RL_SHL:
 	defw RL_SHR
-	defb 053h
-	defb 048h
-	defb 0cch
+	defb 'SH','L'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3999,8 +3556,7 @@
 	defb 003h
 RL_LO:
 	defw RL_SHL
-	defb 04ch
-	defb 0cfh
+	defb 'L','O'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4009,8 +3565,7 @@
 	defb 002h
 RL_HI:
 	defw RL_LO
-	defb 048h
-	defb 0c9h
+	defb 'H','I'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4019,10 +3574,7 @@
 	defb 002h
 RL_SWAP:
 	defw RL_HI
-	defb 053h
-	defb 057h
-	defb 041h
-	defb 0d0h
+	defb 'SWA','P'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4031,10 +3583,7 @@
 	defb 002h
 RL_BXOR:
 	defw RL_SWAP
-	defb 042h
-	defb 058h
-	defb 04fh
-	defb 0d2h
+	defb 'BXO','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4043,9 +3592,7 @@
 	defb 003h
 RL_BOR:
 	defw RL_BXOR
-	defb 042h
-	defb 04fh
-	defb 0d2h
+	defb 'BO','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4054,10 +3601,7 @@
 	defb 003h
 RL_BAND:
 	defw RL_BOR
-	defb 042h
-	defb 041h
-	defb 04eh
-	defb 0c4h
+	defb 'BAN','D'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4066,53 +3610,37 @@
 	defb 003h
 RL_EXP:
 	defw RL_BAND
-	defb 045h
-	defb 058h
-	defb 0d0h
+	defb 'EX','P'+0x80
 	defb 00bh
 	defw Exp__
 RL_LN:
 	defw RL_EXP
-	defb 04ch
-	defb 0ceh
+	defb 'L','N'+0x80
 	defb 00bh
 	defw Ln__
 RL_ARCTAN:
 	defw RL_LN
-	defb 041h
-	defb 052h
-	defb 043h
-	defb 054h
-	defb 041h
-	defb 0ceh
+	defb 'ARCTA','N'+0x80
 	defb 00bh
 	defw Arctan__
 RL_TAN:
 	defw RL_ARCTAN
-	defb 054h
-	defb 041h
-	defb 0ceh
+	defb 'TA','N'+0x80
 	defb 00bh
 	defw Tan__
 RL_COS:
 	defw RL_TAN
-	defb 043h
-	defb 04fh
-	defb 0d3h
+	defb 'CO','S'+0x80
 	defb 00bh
 	defw Cos__
 RL_SIN:
 	defw RL_COS
-	defb 053h
-	defb 049h
-	defb 0ceh
+	defb 'SI','N'+0x80
 	defb 00bh
 	defw Sin__
 RL_INP:
 	defw RL_SIN
-	defb 049h
-	defb 04eh
-	defb 0d0h
+	defb 'IN','P'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -4121,57 +3649,36 @@
 	defb 002h
 RL_OUT:
 	defw RL_INP
-	defb 04fh
-	defb 055h
-	defb 0d4h
+	defb 'OU','T'+0x80
 	defb 006h
 	defb 090h
 	defb 052h
 RL_SIZE:
 	defw RL_OUT
-	defb 053h
-	defb 049h
-	defb 05ah
-	defb 0c5h
+	defb 'SIZ','E'+0x80
 	defb 007h
 	defb 06bh
 	defb 052h
 RL_ADDR:
 	defw RL_SIZE
-	defb 041h
-	defb 044h
-	defb 044h
-	defb 0d2h
+	defb 'ADD','R'+0x80
 	defb 007h
 	defb 07fh
 	defb 052h
 RL_INLINE:
 	defw RL_ADDR
-	defb 049h
-	defb 04eh
-	defb 04ch
-	defb 049h
-	defb 04eh
-	defb 0c5h
+	defb 'INLIN','E'+0x80
 	defb 006h
 	defb 08ah
 	defb 043h
 RL_ENTIER:
 	defw RL_INLINE
-	defb 045h
-	defb 04eh
-	defb 054h
-	defb 049h
-	defb 045h
-	defb 0d2h
+	defb 'ENTIE','R'+0x80
 	defb 00ch
 	defw Entier__
 RL_USER:
 	defw RL_ENTIER
-	defb 055h
-	defb 053h
-	defb 045h
-	defb 0d2h
+	defb 'USE','R'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4180,12 +3687,7 @@
 	defb 002h
 RL_RANDOM:
 	defw RL_USER
-	defb 052h
-	defb 041h
-	defb 04eh
-	defb 044h
-	defb 04fh
-	defb 0cdh
+	defb 'RANDO','M'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4194,16 +3696,7 @@
 	defb 001h
 RL_KEYPRESSED:
 	defw RL_RANDOM
-	defb 04bh
-	defb 045h
-	defb 059h
-	defb 050h
-	defb 052h
-	defb 045h
-	defb 053h
-	defb 053h
-	defb 045h
-	defb 0c4h
+	defb 'KEYPRESSE','D'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4212,10 +3705,7 @@
 	defb 001h
 RL_HALT:
 	defw RL_KEYPRESSED
-	defb 048h
-	defb 041h
-	defb 04ch
-	defb 0d4h
+	defb 'HAL','T'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4224,10 +3714,7 @@
 	defb 001h
 RL_EOLN:
 	defw RL_HALT
-	defb 045h
-	defb 04fh
-	defb 04ch
-	defb 0ceh
+	defb 'EOL','N'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4236,10 +3723,7 @@
 	defb 001h
 RL_PAGE:
 	defw RL_EOLN
-	defb 050h
-	defb 041h
-	defb 047h
-	defb 0c5h
+	defb 'PAG','E'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4248,137 +3732,89 @@
 	defb 001h
 RL_SQRT:
 	defw RL_PAGE
-	defb 053h
-	defb 051h
-	defb 052h
-	defb 0d4h
+	defb 'SQR','T'+0x80
 	defb 00bh
 	defw Sqrt__
 RL_ROUND:
 	defw RL_SQRT
-	defb 052h
-	defb 04fh
-	defb 055h
-	defb 04eh
-	defb 0c4h
+	defb 'ROUN','D'+0x80
 	defb 00ch
 	defw Round__
 RL_TRUNC:
 	defw RL_ROUND
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 04eh
-	defb 0c3h
+	defb 'TRUN','C'+0x80
 	defb 00ch
 	defw Trunc__
 RL_MAXINT:
 	defw RL_TRUNC
-	defb 04dh
-	defb 041h
-	defb 058h
-	defb 049h
-	defb 04eh
-	defb 0d4h
+	defb 'MAXIN','T'+0x80
 	defb 001h
 	defb 001h
 	defb 000h
 	defw 07fffh
 RL_SUCC:
 	defw RL_MAXINT
-	defb 053h
-	defb 055h
-	defb 043h
-	defb 0c3h
+	defb 'SUC','C'+0x80
 	defb 007h
 	defb 06ch
 	defb 043h
 RL_PRED:
 	defw RL_SUCC
-	defb 050h
-	defb 052h
-	defb 045h
-	defb 0c4h
+	defb 'PRE','D'+0x80
 	defb 007h
 	defb 05bh
 	defb 043h
 RL_ORD:
 	defw RL_PRED
-	defb 04fh
-	defb 052h
-	defb 0c4h
+	defb 'OR','D'+0x80
 	defb 007h
 	defb 04ch
 	defb 043h
 RL_PEEK:
 	defw RL_ORD
-	defb 050h
-	defb 045h
-	defb 045h
-	defb 0cbh
+	defb 'PEE','K'+0x80
 	defb 007h
 	defb 036h
 	defb 045h
 RL_POKE:
 	defw RL_PEEK
-	defb 050h
-	defb 04fh
-	defb 04bh
-	defb 0c5h
+	defb 'POK','E'+0x80
 	defb 006h
 	defb 044h
 	defb 038h
 RL_RELEASE:
 	defw RL_POKE
-	defb 052h
-	defb 045h
-	defb 04ch
-	defb 045h
-	defb 041h
-	defb 053h
-	defb 0c5h
+	defb 'RELEAS','E'+0x80
 	defb 006h
 	defb 07bh
 	defb 050h
 RL_MARK:
 	defw RL_RELEASE
-	defb 04dh
-	defb 041h
-	defb 052h
-	defb 0cbh
+	defb 'MAR','K'+0x80
 	defb 006h
 	defb 076h
 	defb 050h
 RL_NEW:
 	defw RL_MARK
-	defb 04eh
-	defb 045h
-	defb 0d7h
+	defb 'NE','W'+0x80
 	defb 006h
 	defb 09eh
 	defb 050h
 RL_TOUT:
 	defw RL_NEW
-	defb 054h
-	defb 04fh
-	defb 055h
-	defb 0d4h
+	defb 'TOU',T''+0x80
 	defb 006h
 	defb 0fah
 	defb 050h
 RL_TIN:
 	defw RL_TOUT
-	defb 054h
-	defb 049h
-	defb 0ceh
+	defb 'TI','N'+0x80
 	defb 006h
 	defb 0eeh
 	defb 050h
 RL_CHR:
 	defw RL_TIN
-	defb 043h
-	defb 048h
-	defb 0d2h
+	defb 'CH',R''+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -4387,9 +3823,7 @@
 	defb 002h
 RL_ODD:
 	defw RL_CHR
-	defb 04fh
-	defb 044h
-	defb 0c4h
+	defb 'OD','D'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4398,9 +3832,7 @@
 	defb 002h
 RL_ABS:
 	defw RL_ODD
-	defb 041h
-	defb 042h
-	defb 0d3h
+	defb 'AB','S'+0x80
 	defb 00dh
 	defb 0e3h
 	defb 052h
@@ -4408,9 +3840,7 @@
 	defb 052h
 RL_SQR:
 	defw RL_ABS
-	defb 053h
-	defb 051h
-	defb 0d2h
+	defb 'SQ','R'+0x80
 	defb 00dh
 	defb 0dch
 	defb 052h
@@ -4418,11 +3848,7 @@
 	defb 052h
 RL_FALSE:
 	defw RL_SQR
-	defb 046h
-	defb 041h
-	defb 04ch
-	defb 053h
-	defb 0c5h
+	defb 'FALS','E'+0x80
 	defb 001h
 	defb 004h
 	defb 000h
@@ -4430,10 +3856,7 @@
 	defb 001h
 RL_TRUE:
 	defw RL_FALSE
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 0c5h
+	defb 'TRU','E'+0x80
 	defb 001h
 	defb 004h
 	defb 000h
@@ -4441,13 +3864,7 @@
 	defb 001h
 RL_BOOLEAN:
 	defw RL_TRUE
-	defb 042h
-	defb 04fh
-	defb 04fh
-	defb 04ch
-	defb 045h
-	defb 041h
-	defb 0ceh
+	defb 'BOOLEA','N'+0x80
 	defb 003h
 	defb 004h
 	defb 000h
@@ -4459,10 +3876,7 @@
 	defb 000h
 RL_CHAR:
 	defw RL_BOOLEAN
-	defb 043h
-	defb 048h
-	defb 041h
-	defb 0d2h
+	defb 'CHA','R'+0x80
 	defb 003h
 	defb 003h
 	defb 000h
@@ -4474,10 +3888,7 @@
 	defb 000h
 RL_REAL:
 	defw RL_CHAR
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 0cch
+	defb 'REA','L'+0x80
 	defb 003h
 	defb 002h
 	defb 000h
@@ -4489,13 +3900,7 @@
 	defb 000h
 RL_INTEGER:
 	defw RL_REAL
-	defb 049h
-	defb 04eh
-	defb 054h
-	defb 045h
-	defb 047h
-	defb 045h
-	defb 0d2h
+	defb 'INTEGE','R'+0x80
 	defb 003h
 	defb 001h
 	defb 000h
@@ -4507,2131 +3912,32 @@
 	defb 000h
 RL_READLN:
 	defw RL_INTEGER
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 044h
-	defb 04ch
-	defb 0ceh
+	defb 'READL','N'+0x80
 	defb 006h
 	defb 0e5h
 	defb 03eh
 RL_READ:
 	defw RL_READLN
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 0c4h
+	defb 'REA','D'+0x80
 	defb 006h
 	defb 095h
 	defb 03eh
 RL_WRITELN:
 	defw RL_READ
-	defb 057h
-	defb 052h
-	defb 049h
-	defb 054h
-	defb 045h
-	defb 04ch
-	defb 0ceh
+	defb 'WRITEL','N'+0x80
 	defb 006h
 	defb 06eh
 	defb 03dh
 RL_WRITE:
 	defw RL_WRITELN
-	defb 057h
-	defb 052h
-	defb 049h
-	defb 054h
-	defb 0c5h
+	defb 'WRIT','E'+0x80
 	defb 006h
 	defb 081h
 	defb 03dh
 
 ; BLOCK 'Symtab_open' (start 0x1b35 end 0x2355)
 Symtab_open_start:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l1f40h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l2053h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 2080
 START_PASCAL:
 	call GetRAMEnd
 l2358h:
@@ -7782,44 +5088,44 @@
 	xor a	
 	jr SetHLFirstSrcLine
 cmdTab:
-	defb 042h
+	defb 'B'
 	defw JEndPascal
-	defb 043h
+	defb 'C'
 	defw DoCmdC
-	defb 044h
+	defb 'D'
 	defw DoCmdD
-	defb 045h
+	defb 'E'
 	defw DoCmdE
-	defb 046h
+	defb 'F'
 	defw DoCmdF
-	defb 047h
+	defb 'G'
 	defw DoCmdG
-	defb 049h
+	defb 'I'
 	defw DoCmdI
-	defb 04ch
+	defb 'L'
 	defw DoCmdL
-	defb 04bh
+	defb 'K'
 	defw DoCmdK
-	defb 04dh
+	defb 'M'
 	defw DoCmdM
-	defb 04eh
+	defb 'N'
 	defw DoCmdN
-	defb 04fh
+	defb 'O'
 	defw DoCmdO
-	defb 050h
+	defb 'P'
 	defw DoCmdP
-	defb 052h
+	defb 'R'
 cmdTabRunAddr:
 	defw PasPrgStart
-	defb 053h
+	defb 'S'
 	defw DoCmdS
-	defb 054h
+	defb 'T'
 	defw DoCmdT
-	defb 056h
+	defb 'V'
 	defw DoCmdV
-	defb 058h
+	defb 'X'
 	defw DoCmdX
-	defb 05ah
+	defb 'Z'
 	defw DoCmdZ
 l2ad6h:
 	dec bc	
@@ -7950,16 +5256,7 @@
 GetSrcChr:
 	jp l5491h
 curIdentifier:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 10
 SP_safe:
 	defb 0f0h
 	defb 07fh
@@ -9205,89 +6502,20 @@
 RL_PROCEDURE:
 ResWordsEntry2:
 	defw RL_FUNCTION
-	defb 050h
-	defb 052h
-	defb 04fh
-	defb 043h
-	defb 045h
-	defb 044h
-	defb 055h
-	defb 052h
-	defb 0c5h
+	defb 'PROCEDUR','E'+080h
 	defb 004h
 ResWordsEntry:
 	defw RL_PROCEDURE
 tEadr:
-	defb 045h
-	defb 06eh
-	defb 064h
-	defb 061h
-	defb 064h
-	defb 072h
-	defb 065h
-	defb 073h
-	defb 073h
-	defb 065h
-	defb 03ah
-	defb 020h
-	defb 000h
+	defb 'Endadresse: ',000h
 tFeh1:
-	defb 046h
-	defb 065h
-	defb 068h
-	defb 06ch
-	defb 065h
-	defb 072h
-	defb 03ah
-	defb 000h
+	defb 'Fehler:',000h
 tFeh2:
-	defb 02ah
-	defb 046h
-	defb 045h
-	defb 048h
-	defb 04ch
-	defb 045h
-	defb 052h
-	defb 02ah
-	defb 000h
+	defb '*FEHLER*',000h
 tKeTx:
-	defb 00dh
-	defb 04bh
-	defb 065h
-	defb 069h
-	defb 06eh
-	defb 020h
-	defb 054h
-	defb 065h
-	defb 078h
-	defb 074h
-	defb 020h
-	defb 06dh
-	defb 065h
-	defb 068h
-	defb 072h
-	defb 021h
-	defb 000h
+	defb 'Kein Text mehr!',000h
 tTabU:
-	defb 054h
-	defb 061h
-	defb 062h
-	defb 065h
-	defb 06ch
-	defb 06ch
-	defb 065h
-	defb 06eh
-	defb 075h
-	defb 065h
-	defb 062h
-	defb 065h
-	defb 072h
-	defb 06ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 021h
-	defb 000h
+	defb 'Tabellenueberlauf!',000h
 ChkType0004:
 	ld de,00004h
 	jr ChkType__
@@ -14750,116 +11978,17 @@
 
 ; BLOCK 'Rest' (start 0x5490 end 0x5500)
 Rest_first:
-	defb 00dh
-l5491h:
-	defb 00ah
-	defb 009h
-	defb 043h
-	defb 050h
-	defb 009h
-	defb 037h
-	defb 046h
-	defb 048h
-	defb 009h
-	defb 03bh
-	defb 054h
-	defb 065h
-	defb 073h
-	defb 074h
-	defb 020h
-	defb 050h
-	defb 072h
-	defb 06fh
-	defb 06ch
-	defb 06fh
-	defb 067h
-	defb 03fh
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 049h
-	defb 04eh
-	defb 043h
-	defb 009h
-	defb 048h
-	defb 04ch
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 04ah
-	defb 052h
-	defb 009h
-	defb 05ah
-	defb 02ch
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 031h
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 044h
-	defb 04ah
-	defb 04eh
-	defb 05ah
-	defb 009h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 032h
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 04ah
-	defb 052h
-	defb 009h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 033h
-	defb 00dh
-	defb 00ah
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 031h
-	defb 009h
-	defb 04ch
-	defb 044h
-	defb 009h
-	defb 04dh
-	defb 02ch
-	defb 030h
-	defb 009h
-	defb 03bh
-	defb 06ch
-	defb 07ch
-	defb 073h
-	defb 063h
-	defb 068h
-	defb 065h
-	defb 06eh
-	defb 00dh
-	defb 00ah
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 033h
-	defb 009h
-	defb 04ch
-	defb 044h
-	defb 009h
-	defb 048h
-	defb 04ch
-	defb 02ch
-	defb 028h
-	defb 043h
-	defb 043h
-	defb 054h
-	defb 04ch
-	defb 030h
+	defb 00dh,00ah,009h,043h,050h,009h,037h,046h ; ...CP.7F 
+	defb 048h,009h,03bh,054h,065h,073h,074h,020h ; H.;Test  
+	defb 050h,072h,06fh,06ch,06fh,067h,03fh,00dh ; Prolog?. 
+	defb 00ah,009h,049h,04eh,043h,009h,048h,04ch ; ..INC.HL 
+	defb 00dh,00ah,009h,04ah,052h,009h,05ah,02ch ; ...JR.Z, 
+	defb 050h,041h,053h,030h,031h,00dh,00ah,009h ; PAS01... 
+	defb 044h,04ah,04eh,05ah,009h,050h,041h,053h ; DJNZ.PAS 
+	defb 030h,032h,00dh,00ah,009h,04ah,052h,009h ; 02...JR. 
+	defb 050h,041h,053h,030h,033h,00dh,00ah,050h ; PAS03..P 
+	defb 041h,053h,030h,031h,009h,04ch,044h,009h ; AS01.LD. 
+	defb 04dh,02ch,030h,009h,03bh,06ch,07ch,073h ; M,0.;l|s 
+	defb 063h,068h,065h,06eh,00dh,00ah,050h,041h ; chen..PA 
+	defb 053h,030h,033h,009h,04ch,044h,009h,048h ; S03.LD.H 
+	defb 04ch,02ch,028h,043h,043h,054h,04ch,030h ; L,(CCTL0 
