--- p51.asm	2025-02-03 18:09:13.263706093 +0100
+++ pascal51.asm	2025-02-03 18:11:09.377798616 +0100
@@ -2,11 +2,12 @@
 ; command line: z80dasm -b p51.block -g 0x200 -l -S p51.sym -s p51.symout -u -o pascal51.asm pascal51_kcc.bin
 
 	org	00200h
-iobuf+11:	equ 0x008b
-iobuf+16:	equ 0x0090
-iobuf+17:	equ 0x0091
-iobuf+19:	equ 0x0093
-iobuf+20:	equ 0x0094
+iobuf:	equ 0x0080
+;iobuf+11:	equ 0x008b
+;iobuf+16:	equ 0x0090
+;iobuf+17:	equ 0x0091
+;iobuf+19:	equ 0x0093
+;iobuf+20:	equ 0x0094
 CAOS_ARG1:	equ 0xb782
 CAOS_ARG2:	equ 0xb784
 CAOS_ARG3:	equ 0xb786
@@ -20,12 +21,12 @@
 CAOS_FARB:	equ 0xb7d6
 SUBALT:	equ 0xb7fe
 PXSASCI:	equ 0xbc00
-PXSASCI+1:	equ 0xbc01
+;PXSASCI+1:	equ 0xbc01
 PXASCI:	equ 0xbc08
 PXTAPE:	equ 0xbc0f
-PXTAPE+1:	equ 0xbc10
+;PXTAPE+1:	equ 0xbc10
 PXDISK:	equ 0xbc20
-PXDISK+1:	equ 0xbc21
+;PXDISK+1:	equ 0xbc21
 ISRO:	equ 0xbc2f
 PXNxtBlock:	equ 0xbc47
 PXSendCtl:	equ 0xbc5c
@@ -36,46 +37,24 @@
 MBI:	equ 0xbcd0
 CSRI:	equ 0xbce9
 SUBNEU:	equ 0xbcee
-SUBNEU+2:	equ 0xbcf0
-SUBNEU+10:	equ 0xbcf8
-SUBNEU+16:	equ 0xbcfe
+;SUBNEU+2:	equ 0xbcf0
+;SUBNEU+10:	equ 0xbcf8
+;SUBNEU+16:	equ 0xbcfe
 PV1:	equ 0xf003
 
 PasPrgMenuHdr:
 	defw 00000h
 PasPrgMenuName:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l0208h:
-	defb 000h
-	defb 000h
+	defs 8
 	defb 000h
 	jp PasPrgStart
 MenuPRec:
 	defw 07f7fh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 052h
-	defb 045h
-	defb 043h
-	defb 000h
+	defb 'PASREC',000h
 	jp Recall
 MenuPEnt:
 	defw 07f7fh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 045h
-	defb 04eh
-	defb 054h
-	defb 052h
-	defb 059h
-	defb 000h
+	defb 'PASENTRY',000h
 	jp Entry
 DoNopRET:
 	nop	
@@ -205,7 +184,6 @@
 	ld (bin_end),de
 	ld (iobuf+19),de
 	ld a,002h
-l0314h:
 	ld (iobuf+16),a
 	ld a,(fileExt)
 	cp 043h
@@ -248,7 +226,7 @@
 	res 5,a
 	res 2,a
 	out (088h),a
-	ld bc,l1f40h
+	ld bc,01f40h
 	ld e,008h
 	call CCaos
 SaveBlock:
@@ -261,7 +239,6 @@
 	ld e,001h
 	call CCaos
 	in a,(088h)
-l0390h:
 	set 5,a
 	set 2,a
 	out (088h),a
@@ -328,7 +305,7 @@
 	inc a	
 	ld (nextBlkNum),a
 	ld de,00080h
-	ld bc,l0b00h
+	ld bc,00b00h
 	ld hl,fileName
 CmpFNamChr:
 	ld a,(de)	
@@ -421,7 +398,6 @@
 	jp CCaos
 CAOS_UOT1:
 	push af	
-l04dch:
 	ld a,c	
 	push de	
 	ld e,002h
@@ -479,7 +455,6 @@
 	and 0f8h
 	ld (CAOS_FARB),a
 	in a,(088h)
-l0545h:
 	res 5,a
 	res 2,a
 	out (088h),a
@@ -724,18 +699,9 @@
 JCompileToRuntimeEnd:
 	jp SetBinStartToRuntimeEnd
 fileName:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h,000h,000h,000h,000h,000h
 fileExt:
-	defb 000h
-	defb 000h
-	defb 000h
+	defb 000h,000h,000h
 	defb 000h
 l0704h:
 	defw 00000h
@@ -847,13 +813,9 @@
 	ld hl,fileExtCOM
 	jr SetFileExt
 fileExtCOM:
-	defb 043h
-	defb 04fh
-	defb 04dh
+	defb 'COM'
 fileExtPAS:
-	defb 050h
-	defb 041h
-	defb 053h
+	defb 'PAS'
 SaveSrcFile:
 	push ix
 	push iy
@@ -1053,21 +1015,12 @@
 
 ; BLOCK 'TFalse' (start 0x08d4 end 0x08da)
 TFalse_start:
-	defb 046h
-	defb 041h
-	defb 04ch
-	defb 053h
-	defb 045h
-	defb 000h
+	defb 'FALSE',000h
 tTrue:
 
 ; BLOCK 'TTrue' (start 0x08da end 0x08df)
 TTrue_start:
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 045h
-	defb 000h
+	defb 'TRUE',000h
 PrRAM:
 	ld de,T_RAM
 	jr PrErr2
@@ -1126,139 +1079,25 @@
 ; BLOCK 'TErrMsg' (start 0x0934 end 0x09b1)
 TErrMsg_start:
 	defb 00dh
-	defb 048h
-	defb 061h
-	defb 06ch
-	defb 074h
+	defb 'Halt'
 T_beiPC:
-	defb 020h
-	defb 062h
-	defb 065h
-	defb 069h
-	defb 020h
-	defb 050h
-	defb 043h
-	defb 03dh
-	defb 000h
+	defb ' bei PC=',000h
 TUeber:
-	defb 055h
-	defb 065h
-	defb 062h
-	defb 065h
-	defb 072h
-	defb 06ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 000h
+	defb 'Ueberlauf',000h
 T_RAM:
-	defb 03eh
-	defb 03eh
-	defb 020h
-	defb 052h
-	defb 041h
-	defb 04dh
-	defb 000h
+	defb '>> RAM',000h
 TDiv0:
-	defb 02fh
-	defb 020h
-	defb 064h
-	defb 075h
-	defb 072h
-	defb 063h
-	defb 068h
-	defb 020h
-	defb 04eh
-	defb 075h
-	defb 06ch
-	defb 06ch
-	defb 000h
+	defb '/ durch Null',000h
 TIdxLow:
-	defb 049h
-	defb 06eh
-	defb 064h
-	defb 065h
-	defb 078h
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 06eh
-	defb 069h
-	defb 065h
-	defb 064h
-	defb 072h
-	defb 069h
-	defb 067h
-	defb 000h
+	defb 'Index zu niedrig',000h
 TIdxHigh:
-	defb 049h
-	defb 06eh
-	defb 064h
-	defb 065h
-	defb 078h
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 068h
-	defb 06fh
-	defb 063h
-	defb 068h
-	defb 000h
+	defb 'Index zu hoch',000h
 TMathErr:
-	defb 06dh
-	defb 061h
-	defb 074h
-	defb 068h
-	defb 065h
-	defb 06dh
-	defb 061h
-	defb 074h
-	defb 069h
-	defb 073h
-	defb 063h
-	defb 068h
-	defb 065h
-	defb 072h
-	defb 020h
-	defb 046h
-	defb 065h
-	defb 068h
-	defb 06ch
-	defb 065h
-	defb 072h
-	defb 000h
+	defb 'mathematischer Fehler',000h
 TNumTooBig:
-	defb 05ah
-	defb 061h
-	defb 068h
-	defb 06ch
-	defb 020h
-	defb 07ah
-	defb 075h
-	defb 020h
-	defb 067h
-	defb 072h
-	defb 06fh
-	defb 07eh
-	defb 000h
-	defb 000h
+	defb 'Zahl zu gro',07eh,000h,000h
 TNumExpected:
-	defb 05ah
-	defb 061h
-	defb 068h
-	defb 06ch
-	defb 020h
-	defb 065h
-	defb 072h
-	defb 077h
-	defb 061h
-	defb 072h
-	defb 074h
-	defb 065h
-	defb 074h
-	defb 000h
+	defb 'Zahl erwartet',000h
 SelListElem__:
 	ld d,ixh
 	ld e,ixl
@@ -1516,7 +1355,6 @@
 	jr LineEdLoop__
 TestCuD:
 	cp 010h
-l0b00h:
 	jr nz,TestEnter
 	call OutChr
 	jr LineEdNextLine__
@@ -2288,7 +2126,7 @@
 	ret c	
 	push hl	
 	push de	
-	ld bc,l5000h
+	ld bc,05000h
 	jr nz,l0f4ch
 	or a	
 	sbc hl,bc
@@ -2413,7 +2251,7 @@
 	call sub_0e45h
 	call sub_0ca1h
 l1022h:
-	ld bc,l5000h
+	ld bc,05000h
 	push bc	
 	ld b,003h
 	push bc	
@@ -2559,7 +2397,6 @@
 	ld a,045h
 	call OutChr
 	ld a,(l1797h)
-l1110h:
 	or a	
 	jp p,l111bh
 	neg
@@ -2611,7 +2448,7 @@
 	ld a,d	
 	or a	
 	jp p,l10bdh
-	ld hl,l5000h
+	ld hl,05000h
 	ld de,00300h
 	call RealMul__
 	push hl	
@@ -2644,15 +2481,8 @@
 	defb 0e2h
 	defb 053h
 realStrPart:
-	defb 020h
-	defb 030h
-	defb 02eh
-	defb 000h
-	defb 045h
-	defb 02bh
-	defb 030h
-	defb 030h
-	defb 000h
+	defb ' 0.',000h
+        defb 'E+00',000h
 Sqrt__:
 	ld a,h	
 	or a	
@@ -2842,24 +2672,7 @@
 	ld de,tExpErw
 	jp PrErr2
 tExpErw:
-	defb 045h
-	defb 078h
-	defb 070h
-	defb 06fh
-	defb 06eh
-	defb 065h
-	defb 06eh
-	defb 074h
-	defb 020h
-	defb 065h
-	defb 072h
-	defb 077h
-	defb 061h
-	defb 072h
-	defb 074h
-	defb 065h
-	defb 074h
-	defb 000h
+	defb 'Exponent erwartet',000h
 Frac__:
 	ld a,h	
 	or a	
@@ -2956,11 +2769,11 @@
 	push de	
 	ld bc,0c53fh
 	push bc	
-	ld bc,C_ISRI
+	ld bc,003d6h
 	push bc	
 	ld bc,063e7h
 	push bc	
-	ld bc,l04dch
+	ld bc,004dch
 	push bc	
 	call RealDiv__
 	call sub_0ca1h
@@ -3017,27 +2830,27 @@
 	ld a,d	
 	ld (l1797h),a
 	ld d,000h
-	ld bc,sub_4072h+1
+	ld bc,04073h
 	push bc	
-	ld bc,l02a1h
+	ld bc,002a1h
 	push bc	
 	ld bc,0c4d2h
 	push bc	
-	ld bc,l0545h
+	ld bc,00545h
 	push bc	
 	push hl	
 	push de	
-	ld bc,l5309h
+	ld bc,05309h
 	push bc	
-	ld bc,l0390h
+	ld bc,00390h
 	push bc	
 	ld bc,0c103h
 	push bc	
-	ld bc,l0314h
+	ld bc,00314h
 	push bc	
 	push hl	
 	push de	
-	ld bc,l41a3h
+	ld bc,041a3h
 	push bc	
 	ld bc,00189h
 	push bc	
@@ -3402,7 +3215,7 @@
 	push de	
 	add a,a	
 	add a,a	
-	ld hl,01665h
+	ld hl,xxTab3__-4
 	ld e,a	
 	ld d,000h
 	add hl,de	
@@ -3478,7 +3291,7 @@
 	call sub_0ca1h
 	push hl	
 	push de	
-	ld bc,l4305h
+	ld bc,04305h
 	push bc	
 	ld bc,0ff49h
 	push bc	
@@ -3547,7 +3360,7 @@
 	push bc	
 	push hl	
 	push de	
-	ld bc,l52aeh
+	ld bc,052aeh
 	push bc	
 	ld bc,003f5h
 	push bc	
@@ -3562,47 +3375,9 @@
 	call RealMul__
 	jp l150eh
 l1761h:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 8
 l1769h:
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
-	nop	
+	defs 32
 retAddr__:
 	defw 00000h
 l178bh:
@@ -3639,175 +3414,11 @@
 iBufCurChrAddr:
 	defw 00000h
 inputBuf:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 85
 RuntimeEnd:
 	nop	
 lineBuf:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l1819h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 80
 lineBufEnd:
 	defb 00dh
 stack_adr:
@@ -3819,23 +3430,13 @@
 	defw START_PASCAL
 RL_SETSYS:
 	defw 00000h
-	defb 053h
-	defb 045h
-	defb 054h
-	defb 053h
-	defb 059h
-	defb 0d3h
+	defb 'SETSY','S'+0x80
 	defb 006h
 	defb 04ah
 	defb 051h
 RL_GETSYS:
 	defw RL_SETSYS
-	defb 047h
-	defb 045h
-	defb 054h
-	defb 053h
-	defb 059h
-	defb 0d3h
+	defb 'GETSY','S'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3844,32 +3445,19 @@
 	defb 002h
 RL_PLOT:
 	defw RL_GETSYS
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'PLO','T'+0x80
 	defb 006h
 	defb 074h
 	defb 051h
 RL_CLRPLOT:
 	defw RL_PLOT
-	defb 043h
-	defb 04ch
-	defb 052h
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'CLRPLO','T'+0x80
 	defb 006h
 	defb 0a0h
 	defb 051h
 RL_PTEST:
 	defw RL_CLRPLOT
-	defb 050h
-	defb 054h
-	defb 045h
-	defb 053h
-	defb 0d4h
+	defb 'PTES','T'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -3878,10 +3466,7 @@
 	defb 003h
 RL_GETC:
 	defw RL_PTEST
-	defb 047h
-	defb 045h
-	defb 054h
-	defb 0c3h
+	defb 'GET','C'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3890,52 +3475,31 @@
 	defb 003h
 RL_SETC:
 	defw RL_GETC
-	defb 053h
-	defb 045h
-	defb 054h
-	defb 0c3h
+	defb 'SET','C'+0x80
 	defb 006h
 	defb 060h
 	defb 051h
 RL_LINEPLOT:
 	defw RL_SETC
-	defb 04ch
-	defb 049h
-	defb 04eh
-	defb 045h
-	defb 050h
-	defb 04ch
-	defb 04fh
-	defb 0d4h
+	defb 'LINEPLO','T'+0x80
 	defb 006h
 	defb 0b4h
 	defb 051h
 RL_CIRCLE:
 	defw RL_LINEPLOT
-	defb 043h
-	defb 049h
-	defb 052h
-	defb 043h
-	defb 04ch
-	defb 0c5h
+	defb 'CIRCL','E'+0x80
 	defb 006h
 	defb 0d0h
 	defb 051h
 RL_GOTOXY:
 	defw RL_CIRCLE
-	defb 047h
-	defb 04fh
-	defb 054h
-	defb 04fh
-	defb 058h
-	defb 0d9h
+	defb 'GOTOX','Y'+0x80
 	defb 006h
 	defb 036h
 	defb 051h
 RL_PI:
 	defw RL_GOTOXY
-	defb 050h
-	defb 0c9h
+	defb 'P','I'+0x80
 	defb 001h
 	defb 002h
 	defb 000h
@@ -3945,21 +3509,12 @@
 	defb 064h
 RL_FRAC:
 	defw RL_PI
-	defb 046h
-	defb 052h
-	defb 041h
-	defb 0c3h
+	defb 'FRA','C'+0x80
 	defb 00bh
 	defw Frac__
 RL_READKBD:
 	defw RL_FRAC
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 044h
-	defb 04bh
-	defb 042h
-	defb 0c4h
+	defb 'READKB','D'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -3968,9 +3523,7 @@
 	defb 001h
 RL_SHR:
 	defw RL_READKBD
-	defb 053h
-	defb 048h
-	defb 0d2h
+	defb 'SH','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3979,9 +3532,7 @@
 	defb 003h
 RL_SHL:
 	defw RL_SHR
-	defb 053h
-	defb 048h
-	defb 0cch
+	defb 'SH','L'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -3990,8 +3541,7 @@
 	defb 003h
 RL_LO:
 	defw RL_SHL
-	defb 04ch
-	defb 0cfh
+	defb 'L','O'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4000,8 +3550,7 @@
 	defb 002h
 RL_HI:
 	defw RL_LO
-	defb 048h
-	defb 0c9h
+	defb 'H','I'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4010,10 +3559,7 @@
 	defb 002h
 RL_SWAP:
 	defw RL_HI
-	defb 053h
-	defb 057h
-	defb 041h
-	defb 0d0h
+	defb 'SWA','P'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4022,10 +3568,7 @@
 	defb 002h
 RL_BXOR:
 	defw RL_SWAP
-	defb 042h
-	defb 058h
-	defb 04fh
-	defb 0d2h
+	defb 'BXO','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4034,9 +3577,7 @@
 	defb 003h
 RL_BOR:
 	defw RL_BXOR
-	defb 042h
-	defb 04fh
-	defb 0d2h
+	defb 'BO','R'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4045,10 +3586,7 @@
 	defb 003h
 RL_BAND:
 	defw RL_BOR
-	defb 042h
-	defb 041h
-	defb 04eh
-	defb 0c4h
+	defb 'BAN','D'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4057,53 +3595,37 @@
 	defb 003h
 RL_EXP:
 	defw RL_BAND
-	defb 045h
-	defb 058h
-	defb 0d0h
+	defb 'EX','P'+0x80
 	defb 00bh
 	defw Exp__
 RL_LN:
 	defw RL_EXP
-	defb 04ch
-	defb 0ceh
+	defb 'L','N'+0x80
 	defb 00bh
 	defw Ln__
 RL_ARCTAN:
 	defw RL_LN
-	defb 041h
-	defb 052h
-	defb 043h
-	defb 054h
-	defb 041h
-	defb 0ceh
+	defb 'ARCTA','N'+0x80
 	defb 00bh
 	defw Arctan__
 RL_TAN:
 	defw RL_ARCTAN
-	defb 054h
-	defb 041h
-	defb 0ceh
+	defb 'TA','N'+0x80
 	defb 00bh
 	defw Tan__
 RL_COS:
 	defw RL_TAN
-	defb 043h
-	defb 04fh
-	defb 0d3h
+	defb 'CO','S'+0x80
 	defb 00bh
 	defw Cos__
 RL_SIN:
 	defw RL_COS
-	defb 053h
-	defb 049h
-	defb 0ceh
+	defb 'SI','N'+0x80
 	defb 00bh
 	defw Sin__
 RL_INP:
 	defw RL_SIN
-	defb 049h
-	defb 04eh
-	defb 0d0h
+	defb 'IN','P'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -4112,57 +3634,36 @@
 	defb 002h
 RL_OUT:
 	defw RL_INP
-	defb 04fh
-	defb 055h
-	defb 0d4h
+	defb 'OU','T'+0x80
 	defb 006h
 	defb 090h
 	defb 052h
 RL_SIZE:
 	defw RL_OUT
-	defb 053h
-	defb 049h
-	defb 05ah
-	defb 0c5h
+	defb 'SIZ','E'+0x80
 	defb 007h
 	defb 06bh
 	defb 052h
 RL_ADDR:
 	defw RL_SIZE
-	defb 041h
-	defb 044h
-	defb 044h
-	defb 0d2h
+	defb 'ADD','R'+0x80
 	defb 007h
 	defb 07fh
 	defb 052h
 RL_INLINE:
 	defw RL_ADDR
-	defb 049h
-	defb 04eh
-	defb 04ch
-	defb 049h
-	defb 04eh
-	defb 0c5h
+	defb 'INLIN','E'+0x80
 	defb 006h
 	defb 08ah
 	defb 043h
 RL_ENTIER:
 	defw RL_INLINE
-	defb 045h
-	defb 04eh
-	defb 054h
-	defb 049h
-	defb 045h
-	defb 0d2h
+	defb 'ENTIE','R'+0x80
 	defb 00ch
 	defw Entier__
 RL_USER:
 	defw RL_ENTIER
-	defb 055h
-	defb 053h
-	defb 045h
-	defb 0d2h
+	defb 'USE','R'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4171,12 +3672,7 @@
 	defb 002h
 RL_RANDOM:
 	defw RL_USER
-	defb 052h
-	defb 041h
-	defb 04eh
-	defb 044h
-	defb 04fh
-	defb 0cdh
+	defb 'RANDO','M'+0x80
 	defb 009h
 	defb 001h
 	defb 000h
@@ -4185,16 +3681,7 @@
 	defb 001h
 RL_KEYPRESSED:
 	defw RL_RANDOM
-	defb 04bh
-	defb 045h
-	defb 059h
-	defb 050h
-	defb 052h
-	defb 045h
-	defb 053h
-	defb 053h
-	defb 045h
-	defb 0c4h
+	defb 'KEYPRESSE','D'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4203,10 +3690,7 @@
 	defb 001h
 RL_HALT:
 	defw RL_KEYPRESSED
-	defb 048h
-	defb 041h
-	defb 04ch
-	defb 0d4h
+	defb 'HAL','T'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4215,10 +3699,7 @@
 	defb 001h
 RL_EOLN:
 	defw RL_HALT
-	defb 045h
-	defb 04fh
-	defb 04ch
-	defb 0ceh
+	defb 'EOL','N'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4227,10 +3708,7 @@
 	defb 001h
 RL_PAGE:
 	defw RL_EOLN
-	defb 050h
-	defb 041h
-	defb 047h
-	defb 0c5h
+	defb 'PAG','E'+0x80
 	defb 008h
 	defb 000h
 	defb 000h
@@ -4239,137 +3717,89 @@
 	defb 001h
 RL_SQRT:
 	defw RL_PAGE
-	defb 053h
-	defb 051h
-	defb 052h
-	defb 0d4h
+	defb 'SQR','T'+0x80
 	defb 00bh
 	defw Sqrt__
 RL_ROUND:
 	defw RL_SQRT
-	defb 052h
-	defb 04fh
-	defb 055h
-	defb 04eh
-	defb 0c4h
+	defb 'ROUN','D'+0x80
 	defb 00ch
 	defw Round__
 RL_TRUNC:
 	defw RL_ROUND
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 04eh
-	defb 0c3h
+	defb 'TRUN','C'+0x80
 	defb 00ch
 	defw Trunc__
 RL_MAXINT:
 	defw RL_TRUNC
-	defb 04dh
-	defb 041h
-	defb 058h
-	defb 049h
-	defb 04eh
-	defb 0d4h
+	defb 'MAXIN','T'+0x80
 	defb 001h
 	defb 001h
 	defb 000h
 	defw 07fffh
 RL_SUCC:
 	defw RL_MAXINT
-	defb 053h
-	defb 055h
-	defb 043h
-	defb 0c3h
+	defb 'SUC','C'+0x80
 	defb 007h
 	defb 06ch
 	defb 043h
 RL_PRED:
 	defw RL_SUCC
-	defb 050h
-	defb 052h
-	defb 045h
-	defb 0c4h
+	defb 'PRE','D'+0x80
 	defb 007h
 	defb 05bh
 	defb 043h
 RL_ORD:
 	defw RL_PRED
-	defb 04fh
-	defb 052h
-	defb 0c4h
+	defb 'OR','D'+0x80
 	defb 007h
 	defb 04ch
 	defb 043h
 RL_PEEK:
 	defw RL_ORD
-	defb 050h
-	defb 045h
-	defb 045h
-	defb 0cbh
+	defb 'PEE','K'+0x80
 	defb 007h
 	defb 036h
 	defb 045h
 RL_POKE:
 	defw RL_PEEK
-	defb 050h
-	defb 04fh
-	defb 04bh
-	defb 0c5h
+	defb 'POK','E'+0x80
 	defb 006h
 	defb 044h
 	defb 038h
 RL_RELEASE:
 	defw RL_POKE
-	defb 052h
-	defb 045h
-	defb 04ch
-	defb 045h
-	defb 041h
-	defb 053h
-	defb 0c5h
+	defb 'RELEAS','E'+0x80
 	defb 006h
 	defb 07bh
 	defb 050h
 RL_MARK:
 	defw RL_RELEASE
-	defb 04dh
-	defb 041h
-	defb 052h
-	defb 0cbh
+	defb 'MAR','K'+0x80
 	defb 006h
 	defb 076h
 	defb 050h
 RL_NEW:
 	defw RL_MARK
-	defb 04eh
-	defb 045h
-	defb 0d7h
+	defb 'NE','W'+0x80
 	defb 006h
 	defb 09eh
 	defb 050h
 RL_TOUT:
 	defw RL_NEW
-	defb 054h
-	defb 04fh
-	defb 055h
-	defb 0d4h
+	defb 'TOU','T'+0x80
 	defb 006h
 	defb 0fah
 	defb 050h
 RL_TIN:
 	defw RL_TOUT
-	defb 054h
-	defb 049h
-	defb 0ceh
+	defb 'TI','N'+0x80
 	defb 006h
 	defb 0eeh
 	defb 050h
 RL_CHR:
 	defw RL_TIN
-	defb 043h
-	defb 048h
-	defb 0d2h
+	defb 'CH','R'+0x80
 	defb 009h
 	defb 003h
 	defb 000h
@@ -4378,9 +3808,7 @@
 	defb 002h
 RL_ODD:
 	defw RL_CHR
-	defb 04fh
-	defb 044h
-	defb 0c4h
+	defb 'OD','D'+0x80
 	defb 009h
 	defb 004h
 	defb 000h
@@ -4389,9 +3817,7 @@
 	defb 002h
 RL_ABS:
 	defw RL_ODD
-	defb 041h
-	defb 042h
-	defb 0d3h
+	defb 'AB','S'+0x80
 	defb 00dh
 	defb 0e3h
 	defb 052h
@@ -4399,9 +3825,7 @@
 	defb 052h
 RL_SQR:
 	defw RL_ABS
-	defb 053h
-	defb 051h
-	defb 0d2h
+	defb 'SQ','R'+0x80
 	defb 00dh
 	defb 0dch
 	defb 052h
@@ -4409,11 +3833,7 @@
 	defb 052h
 RL_FALSE:
 	defw RL_SQR
-	defb 046h
-	defb 041h
-	defb 04ch
-	defb 053h
-	defb 0c5h
+	defb 'FALS','E'+0x80
 	defb 001h
 	defb 004h
 	defb 000h
@@ -4421,10 +3841,7 @@
 	defb 001h
 RL_TRUE:
 	defw RL_FALSE
-	defb 054h
-	defb 052h
-	defb 055h
-	defb 0c5h
+	defb 'TRU','E'+0x80
 	defb 001h
 	defb 004h
 	defb 000h
@@ -4432,13 +3849,7 @@
 	defb 001h
 RL_BOOLEAN:
 	defw RL_TRUE
-	defb 042h
-	defb 04fh
-	defb 04fh
-	defb 04ch
-	defb 045h
-	defb 041h
-	defb 0ceh
+	defb 'BOOLEA','N'+0x80
 	defb 003h
 	defb 004h
 	defb 000h
@@ -4450,10 +3861,7 @@
 	defb 000h
 RL_CHAR:
 	defw RL_BOOLEAN
-	defb 043h
-	defb 048h
-	defb 041h
-	defb 0d2h
+	defb 'CHA','R'+0x80
 	defb 003h
 	defb 003h
 	defb 000h
@@ -4465,10 +3873,7 @@
 	defb 000h
 RL_REAL:
 	defw RL_CHAR
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 0cch
+	defb 'REA','L'+0x80
 	defb 003h
 	defb 002h
 	defb 000h
@@ -4480,13 +3885,7 @@
 	defb 000h
 RL_INTEGER:
 	defw RL_REAL
-	defb 049h
-	defb 04eh
-	defb 054h
-	defb 045h
-	defb 047h
-	defb 045h
-	defb 0d2h
+	defb 'INTEGE','R'+0x80
 	defb 003h
 	defb 001h
 	defb 000h
@@ -4498,2130 +3897,33 @@
 	defb 000h
 RL_READLN:
 	defw RL_INTEGER
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 044h
-	defb 04ch
-	defb 0ceh
+	defb 'READL','N'+0x80
 	defb 006h
 	defb 0e5h
 	defb 03eh
 RL_READ:
 	defw RL_READLN
-	defb 052h
-	defb 045h
-	defb 041h
-	defb 0c4h
+	defb 'REA','D'+0x80
 	defb 006h
 	defb 095h
 	defb 03eh
 RL_WRITELN:
 	defw RL_READ
-	defb 057h
-	defb 052h
-	defb 049h
-	defb 054h
-	defb 045h
-	defb 04ch
-	defb 0ceh
+	defb 'WRITEL','N'+0x80
 	defb 006h
 	defb 06eh
 	defb 03dh
 RL_WRITE:
 	defw RL_WRITELN
-	defb 057h
-	defb 052h
-	defb 049h
-	defb 054h
-	defb 0c5h
+	defb 'WRIT','E'+0x80
 	defb 006h
 	defb 081h
 	defb 03dh
 
 ; BLOCK 'Symtab_open' (start 0x1b35 end 0x2355)
 Symtab_open_start:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-l1f40h:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 2080
+
 START_PASCAL:
 	call GetRAMEnd
 l2358h:
@@ -7779,44 +5081,44 @@
 	xor a	
 	jr SetHLFirstSrcLine
 cmdTab:
-	defb 042h
+	defb 'B'
 	defw JEndPascal
-	defb 043h
+	defb 'C'
 	defw DoCmdC
-	defb 044h
+	defb 'D'
 	defw DoCmdD
-	defb 045h
+	defb 'E'
 	defw DoCmdE
-	defb 046h
+	defb 'F'
 	defw DoCmdF
-	defb 047h
+	defb 'G'
 	defw DoCmdG
-	defb 049h
+	defb 'I'
 	defw DoCmdI
-	defb 04ch
+	defb 'L'
 	defw DoCmdL
-	defb 04bh
+	defb 'K'
 	defw DoCmdK
-	defb 04dh
+	defb 'M'
 	defw DoCmdM
-	defb 04eh
+	defb 'N'
 	defw DoCmdN
-	defb 04fh
+	defb 'O'
 	defw DoCmdO
-	defb 050h
+	defb 'P'
 	defw DoCmdP
-	defb 052h
+	defb 'R'
 cmdTabRunAddr:
 	defw PasPrgStart
-	defb 053h
+	defb 'S'
 	defw DoCmdS
-	defb 054h
+	defb 'T'
 	defw DoCmdT
-	defb 056h
+	defb 'V'
 	defw DoCmdV
-	defb 058h
+	defb 'X'
 	defw DoCmdX
-	defb 05ah
+	defb 'Z'
 	defw DoCmdZ
 EdCmdTab:
 	defb 00bh
@@ -7825,82 +5127,36 @@
 	defw sub_2619h
 	defb 008h
 	defw sub_2668h
-	defb 043h
+	defb 'C'
 	defw l2672h
 	defb 00dh
 	defw l28e8h
-	defb 046h
+	defb 'F'
 	defw l26e8h
-	defb 049h
+	defb 'I'
 	defw l267ch
-	defb 04bh
+	defb 'K'
 	defw l2636h
-	defb 04ch
+	defb 'L'
 	defw sub_29d4h
-	defb 051h
+	defb 'Q'
 	defw l29f5h
-	defb 053h
+	defb 'S'
 	defw l2692h
-	defb 052h
+	defb 'R'
 	defw l2561h
-	defb 058h
+	defb 'X'
 	defw l2677h
-	defb 05ah
+	defb 'Z'
 	defw l2681h
 tPardon:
-	defb 050h
-	defb 061h
-	defb 072h
-	defb 064h
-	defb 06fh
-	defb 06eh
-	defb 03fh
-	defb 00dh
-	defb 000h
+	defb 'Pardon?',00dh,000h
 param3:
 	defb 00dh
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 20
 param4:
 	defb 00dh
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 20
 l2b33h:
 	nop	
 lineNumStart__:
@@ -7942,18 +5198,9 @@
 listPageSize:
 	nop	
 GetSrcChr:
-	jp l5491h
+	jp 05491h
 curIdentifier:
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
-	defb 000h
+	defs 10
 SP_safe:
 	defb 0f0h
 	defb 07fh
@@ -8166,81 +5413,10 @@
 banner_nl:
 	defb 00dh
 banner:
-	defb 00ch
-	defb 012h
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 020h
-	defb 04bh
-	defb 043h
-	defb 02dh
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 043h
-	defb 041h
-	defb 04ch
-	defb 020h
-	defb 056h
-	defb 035h
-	defb 02eh
-	defb 031h
-	defb 020h
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 02ah
-	defb 00dh
-	defb 00dh
-	defb 042h
-	defb 045h
-	defb 041h
-	defb 052h
-	defb 042h
-	defb 02eh
-	defb 020h
-	defb 056h
-	defb 04fh
-	defb 04eh
-	defb 020h
-	defb 020h
-	defb 02bh
-	defb 02bh
-	defb 02bh
-	defb 020h
-	defb 041h
-	defb 04dh
-	defb 039h
-	defb 030h
-	defb 020h
-	defb 02bh
-	defb 02bh
-	defb 02bh
-	defb 00dh
-	defb 00dh
-	defb 020h
-	defb 020h
-	defb 020h
-	defb 028h
-	defb 056h
-	defb 045h
-	defb 052h
-	defb 053h
-	defb 049h
-	defb 04fh
-	defb 04eh
-	defb 020h
-	defb 04bh
-	defb 043h
-	defb 038h
-	defb 035h
-	defb 02fh
-	defb 034h
-	defb 029h
-	defb 00dh
-	defb 000h
+ 	defb 00ch,012h
+ 	defb '**** KC-PASCAL V5.1 ****',00dh,00dh
+ 	defb 'BEARB. VON  +++ AM90 +++',00dh,00dh
+ 	defb '   (VERSION KC85/4)',00dh,000h
 CoVarInitData:
 	defb 020h
 	defw RL_PACKED
@@ -8932,345 +6108,171 @@
 
 ; BLOCK 'compOptTab' (start 0x3165 end 0x3171)
 compOptTab_start:
-	defb 04ch
+	defb 'L'
 	defb 001h
-	defb 04fh
+	defb 'O'
 	defb 002h
-	defb 043h
+	defb 'C'
 	defb 004h
-	defb 053h
+	defb 'S'
 	defb 008h
-	defb 049h
+	defb 'I'
 	defb 010h
-	defb 041h
+	defb 'A'
 	defb 020h
 RL_PACKED:
 	defw 00000h
-	defb 050h
-	defb 041h
-	defb 043h
-	defb 04bh
-	defb 045h
-	defb 0c4h
+	defb 'PACKE','D'+080h
 	defb 023h
 RL_NIL:
 	defw RL_PACKED
-	defb 04eh
-	defb 049h
-	defb 0cch
+	defb 'NI','L'+080h
 	defb 022h
 RL_FORWARD:
 	defw RL_NIL
-	defb 046h
-	defb 04fh
-	defb 052h
-	defb 057h
-	defb 041h
-	defb 052h
-	defb 0c4h
+	defb 'FORWAR','D'+080h
 	defb 01dh
 RL_PROGRAM:
 	defw RL_FORWARD
-	defb 050h
-	defb 052h
-	defb 04fh
-	defb 047h
-	defb 052h
-	defb 041h
-	defb 0cdh
+	defb 'PROGRA','M'+080h
 	defb 001h
 RL_IN:
 	defw RL_PROGRAM
-	defb 049h
-	defb 0ceh
+	defb 'I','N'+080h
 	defb 020h
 RL_OR:
 	defw RL_IN
-	defb 04fh
-	defb 0d2h
+	defb 'O','R'+080h
 	defb 007h
-RL_ON:
+RL_OF:
 	defw RL_OR
-	defb 04fh
-	defb 0c6h
+	defb 'O','F'+080h
 	defb 00bh
 RL_TO:
-	defw RL_ON
-	defb 054h
-	defb 0cfh
+	defw RL_OF
+	defb 'T','O'+080h
 	defb 00ch
 RL_DO:
 	defw RL_TO
-	defb 044h
-	defb 0cfh
+	defb 'D','O'+080h
 	defb 011h
 RL_IF:
 	defw RL_DO
-	defb 049h
-	defb 0c6h
+	defb 'I','F'+080h
 	defb 017h
 RL_SET:
 	defw RL_IF
-	defb 053h
-	defb 045h
-	defb 0d4h
+	defb 'SE','T'+080h
 	defb 01bh
 RL_NOT:
 	defw RL_SET
-	defb 04eh
-	defb 04fh
-	defb 0d4h
+	defb 'NO','T'+080h
 	defb 006h
 RL_MOD:
 	defw RL_NOT
-	defb 04dh
-	defb 04fh
-	defb 0c4h
+	defb 'MO','D'+080h
 	defb 009h
 RL_DIV:
 	defw RL_MOD
-	defb 044h
-	defb 049h
-	defb 0d6h
+	defb 'DI','V'+080h
 	defb 002h
 RL_VAR:
 	defw RL_DIV
-	defb 056h
-	defb 041h
-	defb 0d2h
+	defb 'VA','R'+080h
 	defb 00ah
 RL_AND:
 	defw RL_VAR
-	defb 041h
-	defb 04eh
-	defb 0c4h
+	defb 'AN','D'+080h
 	defb 008h
 RL_FOR:
 	defw RL_AND
-	defb 046h
-	defb 04fh
-	defb 0d2h
+	defb 'FO','R'+080h
 	defb 016h
 RL_END:
 	defw RL_FOR
-	defb 045h
-	defb 04eh
-	defb 0c4h
+	defb 'EN','D'+080h
 	defb 010h
 RL_GOTO:
 	defw RL_END
-	defb 047h
-	defb 04fh
-	defb 054h
-	defb 0cfh
+	defb 'GOT','O'+080h
 	defb 01ah
 RL_WITH:
 	defw RL_GOTO
-	defb 057h
-	defb 049h
-	defb 054h
-	defb 0c8h
+	defb 'WIT','H'+080h
 	defb 019h
 RL_TYPE:
 	defw RL_WITH
-	defb 054h
-	defb 059h
-	defb 050h
-	defb 0c5h
+	defb 'TYP','E'+080h
 	defb 01fh
 RL_CASE:
 	defw RL_TYPE
-	defb 043h
-	defb 041h
-	defb 053h
-	defb 0c5h
+	defb 'CAS','E'+080h
 	defb 014h
 RL_ELSE:
 	defw RL_CASE
-	defb 045h
-	defb 04ch
-	defb 053h
-	defb 0c5h
+	defb 'ELS','E'+080h
 	defb 012h
 RL_THEN:
 	defw RL_ELSE
-	defb 054h
-	defb 048h
-	defb 045h
-	defb 0ceh
+	defb 'THE','N'+080h
 	defb 00eh
 RL_LABEL:
 	defw RL_THEN
-	defb 04ch
-	defb 041h
-	defb 042h
-	defb 045h
-	defb 0cch
+	defb 'LABE','L'+080h
 	defb 021h
 RL_CONST:
 	defw RL_LABEL
-	defb 043h
-	defb 04fh
-	defb 04eh
-	defb 053h
-	defb 0d4h
+	defb 'CONS','T'+080h
 	defb 003h
 RL_ARRAY:
 	defw RL_CONST
-	defb 041h
-	defb 052h
-	defb 052h
-	defb 041h
-	defb 0d9h
+	defb 'ARRA','Y'+080h
 	defb 01ch
 RL_UNTIL:
 	defw RL_ARRAY
-	defb 055h
-	defb 04eh
-	defb 054h
-	defb 049h
-	defb 0cch
+	defb 'UNTI','L'+080h
 	defb 00fh
 RL_WHILE:
 	defw RL_UNTIL
-	defb 057h
-	defb 048h
-	defb 049h
-	defb 04ch
-	defb 0c5h
+	defb 'WHIL','E'+080h
 	defb 015h
 RL_BEGIN:
 	defw RL_WHILE
-	defb 042h
-	defb 045h
-	defb 047h
-	defb 049h
-	defb 0ceh
+	defb 'BEGI','N'+080h
 	defb 018h
 RL_RECORD:
-	defw RL_BEGIN
-	defb 052h
-	defb 045h
-	defb 043h
-	defb 04fh
-	defb 052h
-	defb 0c4h
-	defb 01eh
+ 	defw RL_BEGIN
+	defb 'RECOR','D'+080h
+ 	defb 01eh
 RL_DOWNTO:
 	defw RL_RECORD
-	defb 044h
-	defb 04fh
-	defb 057h
-	defb 04eh
-	defb 054h
-	defb 0cfh
+	defb 'DOWNT','O'+080h
 	defb 00dh
 RL_REPEAT:
-	defw RL_DOWNTO
-	defb 052h
-	defb 045h
-	defb 050h
-	defb 045h
-	defb 041h
-	defb 0d4h
-	defb 013h
+ 	defw RL_DOWNTO
+	defb 'REPEA','T'+080h
+ 	defb 013h
 RL_FUNCTION:
-	defw RL_REPEAT
-	defb 046h
-	defb 055h
-	defb 04eh
-	defb 043h
-	defb 054h
-	defb 049h
-	defb 04fh
-	defb 0ceh
-	defb 005h
+ 	defw RL_REPEAT
+	defb 'FUNCTIO','N'+080h
+ 	defb 005h
 ResWordsEntry2:
 RL_PROCEDURE:
-	defw RL_FUNCTION
-	defb 050h
-	defb 052h
-	defb 04fh
-	defb 043h
-	defb 045h
-	defb 044h
-	defb 055h
-	defb 052h
-	defb 0c5h
-	defb 004h
+ 	defw RL_FUNCTION
+	defb 'PROCEDUR','E'+080h
+ 	defb 004h
 ResWordsEntry:
-	defw ResWordsEntry2
+ 	defw ResWordsEntry2
 tEadr:
-	defb 045h
-	defb 06eh
-	defb 064h
-	defb 061h
-	defb 064h
-	defb 072h
-	defb 065h
-	defb 073h
-	defb 073h
-	defb 065h
-	defb 03ah
-	defb 020h
-	defb 000h
+	defb 'Endadresse: ',000h
 tFeh1:
-	defb 046h
-	defb 065h
-	defb 068h
-	defb 06ch
-	defb 065h
-	defb 072h
-	defb 03ah
-	defb 000h
+	defb 'Fehler:',000h
 tFeh2:
-	defb 02ah
-	defb 046h
-	defb 045h
-	defb 048h
-	defb 04ch
-	defb 045h
-	defb 052h
-	defb 02ah
-	defb 000h
+	defb '*FEHLER*',000h
 tKeTx:
-	defb 00dh
-	defb 04bh
-	defb 065h
-	defb 069h
-	defb 06eh
-	defb 020h
-	defb 054h
-	defb 065h
-	defb 078h
-	defb 074h
-	defb 020h
-	defb 06dh
-	defb 065h
-	defb 068h
-	defb 072h
-	defb 021h
-	defb 000h
+	defb 00dh,'Kein Text mehr!',000h
 tTabU:
-	defb 054h
-	defb 061h
-	defb 062h
-	defb 065h
-	defb 06ch
-	defb 06ch
-	defb 065h
-	defb 06eh
-	defb 075h
-	defb 065h
-	defb 062h
-	defb 065h
-	defb 072h
-	defb 06ch
-	defb 061h
-	defb 075h
-	defb 066h
-	defb 021h
-	defb 000h
+	defb 'Tabellenueberlauf!',000h
 ChkTypeBool__:
 	ld de,00004h
 	jr ChkType__
@@ -9893,7 +6895,6 @@
 l35f0h:
 	defb 008h
 	dec b	
-l35f2h:
 	djnz l35ech
 	inc a	
 l35f5h:
@@ -10153,7 +7154,7 @@
 	add a,a	
 	ld l,a	
 	ld h,000h
-	ld de,0371eh
+	ld de,xxTab4__-00026h
 	add hl,de	
 	ld e,(hl)	
 	inc hl	
@@ -10165,7 +7166,7 @@
 	ret	
 sub_376dh:
 	ex af,af'	
-	ld de,l1110h
+	ld de,01110h
 	call ChkLexem_GetLex
 	call WCaBreak_IfCoCo_C
 l3777h:
@@ -10968,7 +7969,7 @@
 	call WCaBreak_IfCoCo_C
 	call sub_3710h
 	push hl	
-	ld de,l1110h
+	ld de,01110h
 	call ChkLexem_GetLex
 	call l3777h
 	call WrJump
@@ -10981,7 +7982,7 @@
 l3c8ah:
 	call sub_3710h
 	push hl	
-	ld de,l0e0fh
+	ld de,00e0fh
 	call ChkLexem_GetLex
 	call l3777h
 	cp 012h
@@ -11070,7 +8071,7 @@
 	push hl	
 	cp 0ach
 	jr z,l3d3ch
-	ld de,l1110h
+	ld de,01110h
 	call ChkLexem_GetLex
 	call l3777h
 l3d32h:
@@ -11429,7 +8430,7 @@
 	ld hl,CSq_l4184h
 	call WCodeOverLastByte
 l3f78h:
-	ld bc,l360ah
+	ld bc,TabCSq-000beh
 WCode_VarIdxTOS:
 	pop hl	
 	call GetCSqAddrByIdx
@@ -11437,16 +8438,16 @@
 l3f82h:
 	pop de	
 	call ChkType__
-	ld bc,l35f2h
+	ld bc,TabCSq-000d6h
 	jr WCode_VarIdxTOS
 l3f8bh:
 	ld a,e	
 	pop de	
 	call ChkType__
 	bit 4,(ix+000h)
-	ld bc,035dah
+	ld bc,TabCSq-000eeh
 	jr z,WCode_VarIdxTOS
-	ld bc,035e6h
+	ld bc,TabCSq-000e2h
 	jr WCode_VarIdxTOS
 l3f9eh:
 	ld a,e	
@@ -11461,7 +8462,7 @@
 	call sub_343dh
 	ld hl,CSq_l35e8h
 	call WCode
-	ld bc,035feh
+	ld bc,TabCSq-000cah
 	jr WCode_VarIdxTOS
 l3fbbh:
 	pop hl	
@@ -11472,7 +8473,7 @@
 	cp 0f4h
 	ld a,e	
 	jr z,l3fd7h
-	ld bc,l3616h
+	ld bc,TabCSq-000b2h
 	call GetCSqAddrByIdx
 	jp CodeLdBC_0n_from_0x2b8b
 l3fd0h:
@@ -11492,7 +8493,7 @@
 	push hl	
 	ld a,l	
 	cp 0f2h
-	ld bc,035dah
+	ld bc,TabCSq-000eeh
 	ld a,e	
 	jr c,WCode_VarIdxTOS
 	ld e,040h
@@ -11768,7 +8769,6 @@
 l419eh:
 	call l381ah
 	djnz l419eh
-l41a3h:
 	ret	
 l41a4h:
 	inc a	
@@ -11991,7 +8991,6 @@
 	ld e,(hl)	
 	inc hl	
 	ld d,(hl)	
-l4305h:
 	push de	
 	inc hl	
 	ld b,(hl)	
@@ -13169,7 +10168,7 @@
 l4aafh:
 	ld a,c	
 ChkBegin:
-	ld de,l1819h
+	ld de,01819h
 	call ChkLexem_GetLex
 	jp l3cd4h
 WCorrAddr:
@@ -14048,7 +11047,6 @@
 	jp l4e61h
 l4fffh:
 	push hl	
-l5000h:
 	ld d,(ix+001h)
 	push de	
 	ld c,001h
@@ -14205,7 +11203,7 @@
 	jr l50f4h
 sub_5105h:
 	call NextChkOpBra_GetLex
-	ld bc,l0208h
+	ld bc,00208h
 	call sub_3f29h
 sub_510eh:
 	call ChkComma_GetLex
@@ -14519,7 +11517,6 @@
 	ld a,(hl)	
 	cp 07fh
 	inc hl	
-l5309h:
 	jr z,PXKillProlog
 	djnz PXSrchProlog
 	jr PXCpChrMap
@@ -14528,7 +11525,7 @@
 PXCpChrMap:
 	ld hl,(CCTL0)
 	ld de,0ba00h
-	ld bc,PasPrgMenuHdr
+	ld bc,00200h
 	ldir
 	ld hl,PXUSASC
 	ld de,0bbd8h
@@ -14768,116 +11765,17 @@
 
 ; BLOCK 'Rest' (start 0x5490 end 0x5500)
 Rest_first:
-	defb 00dh
-l5491h:
-	defb 00ah
-	defb 009h
-	defb 043h
-	defb 050h
-	defb 009h
-	defb 037h
-	defb 046h
-	defb 048h
-	defb 009h
-	defb 03bh
-	defb 054h
-	defb 065h
-	defb 073h
-	defb 074h
-	defb 020h
-	defb 050h
-	defb 072h
-	defb 06fh
-	defb 06ch
-	defb 06fh
-	defb 067h
-	defb 03fh
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 049h
-	defb 04eh
-	defb 043h
-	defb 009h
-	defb 048h
-	defb 04ch
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 04ah
-	defb 052h
-	defb 009h
-	defb 05ah
-	defb 02ch
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 031h
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 044h
-	defb 04ah
-	defb 04eh
-	defb 05ah
-	defb 009h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 032h
-	defb 00dh
-	defb 00ah
-	defb 009h
-	defb 04ah
-	defb 052h
-	defb 009h
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 033h
-	defb 00dh
-	defb 00ah
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 031h
-	defb 009h
-	defb 04ch
-	defb 044h
-	defb 009h
-	defb 04dh
-	defb 02ch
-	defb 030h
-	defb 009h
-	defb 03bh
-	defb 06ch
-	defb 07ch
-	defb 073h
-	defb 063h
-	defb 068h
-	defb 065h
-	defb 06eh
-	defb 00dh
-	defb 00ah
-	defb 050h
-	defb 041h
-	defb 053h
-	defb 030h
-	defb 033h
-	defb 009h
-	defb 04ch
-	defb 044h
-	defb 009h
-	defb 048h
-	defb 04ch
-	defb 02ch
-	defb 028h
-	defb 043h
-	defb 043h
-	defb 054h
-	defb 04ch
-	defb 030h
+	defb 00dh,00ah,009h,043h,050h,009h,037h,046h ; ...CP.7F 
+	defb 048h,009h,03bh,054h,065h,073h,074h,020h ; H.;Test  
+	defb 050h,072h,06fh,06ch,06fh,067h,03fh,00dh ; Prolog?. 
+	defb 00ah,009h,049h,04eh,043h,009h,048h,04ch ; ..INC.HL 
+	defb 00dh,00ah,009h,04ah,052h,009h,05ah,02ch ; ...JR.Z, 
+	defb 050h,041h,053h,030h,031h,00dh,00ah,009h ; PAS01... 
+	defb 044h,04ah,04eh,05ah,009h,050h,041h,053h ; DJNZ.PAS 
+	defb 030h,032h,00dh,00ah,009h,04ah,052h,009h ; 02...JR. 
+	defb 050h,041h,053h,030h,033h,00dh,00ah,050h ; PAS03..P 
+	defb 041h,053h,030h,031h,009h,04ch,044h,009h ; AS01.LD. 
+	defb 04dh,02ch,030h,009h,03bh,06ch,07ch,073h ; M,0.;l|s 
+	defb 063h,068h,065h,06eh,00dh,00ah,050h,041h ; chen..PA 
+	defb 053h,030h,033h,009h,04ch,044h,009h,048h ; S03.LD.H 
+	defb 04ch,02ch,028h,043h,043h,054h,04ch,030h ; L,(CCTL0 
