--- p5a.asm	2025-02-25 07:20:06.998030291 +0100
+++ pas51x.asm	2025-02-25 21:55:19.165295369 +0100
@@ -2,7 +2,7 @@
 ; command line: z80dasm -b p51.block -g 0x200 -l -S p51.sym -s p51.symout -u -o pascal51.asm pascal51_kcc.bin
 
 	org	00200h
-	defb 'PASCAL5AKCC'
+	defb 'PAS51X  KCC'
 	defs 5
 	defb 3
 	defw PasPrgMenuHdr
@@ -10,11 +10,6 @@
 	defw PasEx
 	defs 0x69
 iobuf:	equ 0x0200
-;iobuf+11:	equ 0x008b
-;iobuf+16:	equ 0x0090
-;iobuf+17:	equ 0x0091
-;iobuf+19:	equ 0x0093
-;iobuf+20:	equ 0x0094
 CAOS_ARG1:	equ 0xb782
 CAOS_ARG2:	equ 0xb784
 CAOS_ARG3:	equ 0xb786
@@ -27,26 +22,6 @@
 CAOS_VERT:	equ 0xb7d5
 CAOS_FARB:	equ 0xb7d6
 SUBALT:	equ 0xb7fe
-;;PXSASCI:	equ 0xbc00
-;PXSASCI+1:	equ 0xbc01
-;;PXASCI:	equ 0xbc08
-;;PXTAPE:	equ 0xbc0f
-;PXTAPE+1:	equ 0xbc10
-;;PXDISK:	equ 0xbc20
-;PXDISK+1:	equ 0xbc21
-;;ISRO:	equ 0xbc2f
-;;PXNxtBlock:	equ 0xbc47
-;;PXSendCtl:	equ 0xbc5c
-;;MBO:	equ 0xbc80
-;;CSRO:	equ 0xbc96
-;;ISRI:	equ 0xbc9e
-;;PXRecvBlock:	equ 0xbcb2
-;;MBI:	equ 0xbcd0
-;;CSRI:	equ 0xbce9
-;;SUBNEU:	equ 0xbcee
-;SUBNEU+2:	equ 0xbcf0
-;SUBNEU+10:	equ 0xbcf8
-;SUBNEU+16:	equ 0xbcfe
 PV1:	equ 0xf003
 
 PasPrgMenuHdr:
@@ -176,9 +151,63 @@
 	call CCaos
 	pop de	
 	pop af	
-	ret	
+	ret
+GetCAOSVer:
+	ld a,(0e011h) ; ab KC 85/4 steht hier Menüwort „BASIC“ 
+	cp 7fh	      ; KC 85/4 ?				 
+	ld a,0	      ; KC 85/2 und KC 85/3 = Version 0.0	 
+	ret nz	      					 
+	ld a,(0edffh) ; CAOS-Versionsnummer                    
+	ret
+SetErrDCV:
+	call GetCAOSVer
+        ld hl,IOErrD1
+	cp 046h
+	jr c,SetErrDCV1
+	inc hl
+	jr z,SetErrDCV1
+	inc hl
+SetErrDCV1:
+	ld (SetErrD1+1),hl
+SetErrDev:
+	push hl
+	push de
+	push bc
+	ld a,(ix+8)  ; ix auf CAOS-Wert
+	and 01ch
+	jr nz,SetErrD1
+SetErrD0:
+	ld hl,IOErrD0
+	jr SetErrDx
+SetErrD1:
+	ld hl,IOErrD1 		; Adr. wird von SetErrDCV überschrieben
+SetErrDx:
+	ld de,ChkIOErr3
+	ldi
+	ldi
+	ldi
+	pop bc
+	pop de
+	pop hl
+;	ret
+IOErrD0:
+	ret
+IOErrD1:
+	ret
+	ret
+	ret nc
+	ret nc
+	ret nc
+ChkIOErr3:
+	ret nc
+ChkIOErr2:
+	ret nc
+ChkIOErr:
+	ret nc
+	jp Reset
 Save:
 	ld ix,(caos_ix)
+	call SetErrDev
 	ld (SaveBlockAdr),hl
 	ld (iobuf+17),hl
 	ld hl,iobuf+11
@@ -190,15 +219,23 @@
 	inc de	
 	ld (bin_end),de
 	ld (iobuf+19),de
+
+	ld a,(param4)		; 4. Param. bei Cmd T = 1 --> Autostart
+	cp '1'
 	ld a,002h
+	jr nz,NoAutoStart
+	inc a
+NoAutoStart:
 	ld (iobuf+16),a
 	ld a,(fileExt)
 	cp 043h
 	jr nz,C_ISRO
-	ld a,d	
-	ld (iobuf+19),a
-	ld hl,l04d5h
-	ld (iobuf+20),hl
+;	ld a,d	
+;	ld (iobuf+19),a
+;	ld hl,l04d5h
+;	ld (iobuf+20),hl
+	ld hl,PasPrgStartA
+	ld (iobuf+21),hl
 	ld hl,07f7fh
 	ld (PasPrgMenuHdr),hl
 	ld hl,00000h
@@ -222,6 +259,23 @@
 	ld bc,0000bh
 	ld de,iobuf
 	ldir
+	ld hl,version 		; Pascal-Version in den Vorblock
+	ld de,iobuf+23
+	ld bc,128-23
+	ld a,0dh
+ver_blk0:
+	cp (hl)
+	jr z,bkl0_rest
+	ldi
+	jr ver_blk0
+bkl0_rest:			; Rest mit 0 füllen
+	ex de,hl
+	ld (hl),0h
+	ld d,h
+	ld e,l
+	inc de
+	dec bc
+	ldir
 	ld hl,iobuf
 	in a,(088h)
 	set 5,a
@@ -234,8 +288,10 @@
 	res 2,a
 	out (088h),a
 	ld bc,01f40h
+	ld hl,fileName
 	ld e,008h
 	call CCaos
+	call ChkIOErr2
 SaveBlock:
 	ld hl,(SaveBlockAdr)
 	ld de,iobuf
@@ -245,6 +301,7 @@
 	ld bc,000a0h
 	ld e,001h
 	call CCaos
+	call ChkIOErr2
 	in a,(088h)
 	set 5,a
 	set 2,a
@@ -265,6 +322,7 @@
 C_CSRO:
 	ld e,009h
 	call CCaos
+	call ChkIOErr2
 	ld a,012h
 	call OutChr
 	jp PrNL
@@ -277,6 +335,8 @@
 	jp Reset
 Load:
 	ld (srcAddr_LoadSave),hl
+	ld ix,(caos_ix)
+	call SetErrDev
 RetryISRI:
 	ld ix,(caos_ix)
 C_ISRI:
@@ -293,9 +353,11 @@
 	res 5,a
 	res 2,a
 	out (088h),a
+	ld hl,fileName
 	ld e,00ah
 	call CCaos
-	jr c,RetryISRI
+	call ChkIOErr
+	jr c,RetryISRI 		; nur noch bei DEV 0
 	in a,(088h)
 	set 5,a
 	set 2,a
@@ -308,7 +370,7 @@
 	ld a,e	
 ChkForBlock1:
 	cp 001h
-	jp nz,RetryISRI
+	jp nz,RetryISRI		; erstmal so lassen
 	inc a	
 	ld (nextBlkNum),a
 	ld de,iobuf
@@ -336,7 +398,7 @@
 	call PrNL
 	ld a,c	
 	or a	
-	jp nz,RetryISRI
+	jp nz,RetryISRI		; erstmal so lassen
 	ld de,(iobuf+17)
 	ld hl,(iobuf+19)
 	or a	
@@ -349,7 +411,8 @@
 	call TestBreak
 	ld e,005h
 	call CCaos
-	jr c,C_MBI
+	call ChkIOErr
+	jr c,C_MBI 		; nur noch bei DEV 0
 	in a,(088h)
 	set 5,a
 	set 2,a
@@ -395,8 +458,10 @@
 	ldir
 	ld e,00bh
 	call CCaos
+	call ChkIOErr3
 	ld a,012h
 	call OutChr
+	call PrNL
 	ld hl,(l06b6h)
 l04d5h:
 	ret	
@@ -422,6 +487,7 @@
 	set 2,a
 	out (088h),a
 	ld (CAOS_CURSO),hl
+IRMoff_ret:
 	in a,(088h)
 	res 5,a
 	res 2,a
@@ -675,6 +741,8 @@
 l06c3h:
 	call SaveCAOS_SP
 	jr Reset
+PasPrgStartA:
+	call IRMoff_ret
 PasPrgStart:
 	jp 07cd5h
 JEndPascal:
@@ -697,8 +765,8 @@
 	jp SrcToLineBuf
 JLdHLSrcEnd:
 	jp LdHLSrcEnd
-sub_06e9h:
-	jp l23e1h
+JGetPar3:
+	jp GetPar3
 	jp ExpandLine
 	jp ExpandLineSrc
 JCompile:
@@ -3949,7 +4017,7 @@
 LdHLSrcEnd:
 	ld hl,(endPASSrc_adr)
 	ret	
-l23e1h:
+GetPar3:
 	ld hl,param3
 	ret	
 l23e5h:
@@ -5371,9 +5439,10 @@
 	defb 00dh
 banner:
  	defb 00ch,012h
- 	defb '**** KC-PASCAL V5.1 ****',00dh,00dh
- 	defb 'BEARB. VON  +++ AM90 +++',00dh,00dh
- 	defb '   (VERSION KC85/4)',00dh,000h
+version:
+ 	defb '**** KC-PASCAL V5.1c ****',00dh,00dh
+ 	defb '  BEARB. VON AM90, DU25',00dh,00dh
+ 	defb '   (VERSION KC85/4/5)',00dh,000h
 CoVarInitData:
 	defb 020h
 	defw RL_PACKED
@@ -5673,7 +5742,7 @@
 	exx	
 	push hl	
 	ex de,hl	
-	call sub_06e9h
+	call JGetPar3
 	ld bc,00008h
 	ldir
 	ex de,hl	
@@ -11522,7 +11591,8 @@
 	ld bc,0fc80h
 	in a,(c)
 	cp 0a7h
-	jp nz,PXASCI
+	ld c,0
+	jr nz,PXInitDev
 	ld bc,SUBNEU-PXTAPE
 	ldir
 	ld a,07fh
@@ -11534,7 +11604,7 @@
 	ld (SUBALT),hl
 	ld de,SUBNEU
 	ld (SUTAB),de
-	ld bc,00092h
+	ld bc,00092h		; abhängig von der CAOS-Version
 	ldir
 	ld hl,MBO
 	ld (SUBNEU+2),hl
@@ -11544,6 +11614,9 @@
 	ld de,SUBNEU+16
 	ld c,008h
 	ldir
+	ld c,4
+PXInitDev:
+	call PXSetDev
 	jp PXASCI
 PXSTAB:
 	defw ISRO
@@ -11571,14 +11644,21 @@
 	defb 'PASTAPE'
 	defb 001h
 	ld hl,(SUBALT)
+	ld c,0
 PXSetSUTAB:
 	ld (SUTAB),hl
+PXSetDev:
+	ld a,(ix+8)
+	and 0e3h
+	or c
+	ld (ix+8),a
 	ret	
 PXDISK:
 	defw 00000h
 	defb 'PASDISK'
 	defb 001h
 	ld hl,SUBNEU
+	ld c,4
 	jr PXSetSUTAB
 ISRO:
 	ld (ix+002h),000h
