--- p5a.asm	2025-02-28 22:33:03.238492794 +0100
+++ pas51x.asm	2025-03-01 21:33:10.692101630 +0100
@@ -2,7 +2,7 @@
 ; command line: z80dasm -b p51.block -g 0x200 -l -S p51.sym -s p51.symout -u -o pascal51.asm pascal51_kcc.bin
 
 	org	00200h
-	defb 'PASCAL5AKCC'
+	defb 'PAS51X  KCC'
 	defs 5
 	defb 3
 	defw PasPrgMenuHdr
@@ -10,11 +10,6 @@
 	defw PasEx
 	defs 0x69
 iobuf:	equ 0x0200
-;iobuf+11:	equ 0x008b
-;iobuf+16:	equ 0x0090
-;iobuf+17:	equ 0x0091
-;iobuf+19:	equ 0x0093
-;iobuf+20:	equ 0x0094
 CAOS_ARG1:	equ 0xb782
 CAOS_ARG2:	equ 0xb784
 CAOS_ARG3:	equ 0xb786
@@ -27,26 +22,6 @@
 CAOS_VERT:	equ 0xb7d5
 CAOS_FARB:	equ 0xb7d6
 SUBALT:	equ 0xb7fe
-;;PXSASCI:	equ 0xbc00
-;PXSASCI+1:	equ 0xbc01
-;;PXASCI:	equ 0xbc08
-;;PXTAPE:	equ 0xbc0f
-;PXTAPE+1:	equ 0xbc10
-;;PXDISK:	equ 0xbc20
-;PXDISK+1:	equ 0xbc21
-;;ISRO:	equ 0xbc2f
-;;PXNxtBlock:	equ 0xbc47
-;;PXSendCtl:	equ 0xbc5c
-;;MBO:	equ 0xbc80
-;;CSRO:	equ 0xbc96
-;;ISRI:	equ 0xbc9e
-;;PXRecvBlock:	equ 0xbcb2
-;;MBI:	equ 0xbcd0
-;;CSRI:	equ 0xbce9
-;;SUBNEU:	equ 0xbcee
-;SUBNEU+2:	equ 0xbcf0
-;SUBNEU+10:	equ 0xbcf8
-;SUBNEU+16:	equ 0xbcfe
 PV1:	equ 0xf003
 
 IRM_ON	MACRO
@@ -166,9 +141,65 @@
 	call CCaos
 	pop de	
 	pop af	
-	ret	
-Save:
+	ret
+GetCAOSVer:
+	ld a,(0e011h) ; ab KC 85/4 steht hier Menüwort „BASIC“ 
+	cp 7fh	      ; KC 85/4 ?				 
+	ld a,0	      ; KC 85/2 und KC 85/3 = Version 0.0	 
+	ret nz	      					 
+	ld a,(0edffh) ; CAOS-Versionsnummer                    
+	ret
+SetErrDCV:
+	call GetCAOSVer
+        ld hl,IOErrD1
+	cp 046h
+	jr c,SetErrDCV1
+	inc hl
+	jr z,SetErrDCV1
+	inc hl
+SetErrDCV1:
+	ld (SetErrD1+1),hl
+	ret
+SetErrDev:
+	push hl
+	push de
+	push bc
 	ld ix,(caos_ix)
+	ld hl,IOErrD0
+	IRM_ON
+	ld e,(ix+8)
+	IRM_OFF
+	ld a,e
+	and 01ch
+	jr z,SetErrDx
+SetErrD1:
+	ld hl,IOErrD1 		; Adr. wird von SetErrDCV überschrieben
+SetErrDx:
+	ld de,ChkIOErr3
+	ldi
+	ldi
+	ldi
+	pop bc
+	pop de
+	pop hl
+;	ret
+IOErrD0:
+	ret
+IOErrD1:
+	ret
+	ret
+	ret nc
+	ret nc
+	ret nc
+ChkIOErr3:
+	ret nc 		; wird von SetErrDev überschrieben
+ChkIOErr2:
+	ret nc 		; wird von SetErrDev überschrieben
+ChkIOErr:
+	ret nc 		; wird von SetErrDev überschrieben
+	jp Reset
+Save:
+	call SetErrDev		; setzt u.a. IX auf (caos_ix)
 	ld (SaveBlockAdr),hl
 	ld (iobuf+17),hl
 	ld hl,iobuf+11
@@ -180,15 +211,23 @@
 	inc de	
 	ld (bin_end),de
 	ld (iobuf+19),de
+
+	ld a,(param4)		; 4. Param. bei Cmd T = 1 --> Autostart
+	cp '1'
 	ld a,002h
+	jr nz,NoAutoStart
+	inc a
+NoAutoStart:
 	ld (iobuf+16),a
 	ld a,(fileExt)
 	cp 043h
 	jr nz,C_ISRO
-	ld a,d	
-	ld (iobuf+19),a
-	ld hl,l04d5h
-	ld (iobuf+20),hl
+;	ld a,d	
+;	ld (iobuf+19),a
+;	ld hl,l04d5h
+;	ld (iobuf+20),hl
+	ld hl,PasPrgStartA
+	ld (iobuf+21),hl
 	ld hl,07f7fh
 	ld (PasPrgMenuHdr),hl
 	ld hl,00000h
@@ -212,14 +251,33 @@
 	ld bc,0000bh
 	ld de,iobuf
 	ldir
+	ld hl,version 		; Pascal-Version in den Vorblock
+	ld de,iobuf+23
+	ld bc,128-23
+	ld a,0dh
+ver_blk0:
+	cp (hl)
+	jr z,bkl0_rest
+	ldi
+	jr ver_blk0
+bkl0_rest:			; Rest mit 0 füllen
+	ex de,hl
+	ld (hl),0h
+	ld d,h
+	ld e,l
+	inc de
+	dec bc
+	ldir
 	ld hl,iobuf
 	IRM_ON
 	ld (ix+005h),l
 	ld (ix+006h),h
 	IRM_OFF
 	ld bc,01f40h
+	ld hl,fileName
 	ld e,008h
 	call CCaos
+	call ChkIOErr2
 SaveBlock:
 	ld hl,(SaveBlockAdr)
 	ld de,iobuf
@@ -229,6 +287,7 @@
 	ld bc,000a0h
 	ld e,001h
 	call CCaos
+	call ChkIOErr2
 	IRM_ON
 	ld e,(ix+002h)
 	IRM_OFF
@@ -243,6 +302,7 @@
 C_CSRO:
 	ld e,009h
 	call CCaos
+	call ChkIOErr2
 	ld a,012h
 	call OutChr
 	jp PrNL
@@ -255,6 +315,7 @@
 	jp Reset
 Load:
 	ld (srcAddr_LoadSave),hl
+	call SetErrDev
 RetryISRI:
 	ld ix,(caos_ix)
 C_ISRI:
@@ -265,18 +326,28 @@
 	ld (ix+005h),l
 	ld (ix+006h),h
 	IRM_OFF
+	ld hl,fileName
 	ld e,00ah
 	call CCaos
-	jr c,RetryISRI
+	call ChkIOErr
+	jr c,RetryISRI 		; nur noch bei DEV 0
 	IRM_ON
 	ld e,(ix+002h)
 	IRM_OFF
 	ld a,e	
 ChkForBlock1:
 	cp 001h
-	jp nz,RetryISRI
+	jp nz,RetryISRI		; erstmal so lassen
 	inc a	
 	ld (nextBlkNum),a
+	
+	IRM_ON
+	ld e,(ix+8)		; welches Geraet?
+	IRM_OFF
+	ld a,e
+	and 01ch
+	jr nz,SkipCmpFNam	; kein Tape --> kein Vergleich des Dateinamens
+
 	ld de,iobuf
 	ld bc,00b00h
 	ld hl,fileName
@@ -302,7 +373,9 @@
 	call PrNL
 	ld a,c	
 	or a	
-	jp nz,RetryISRI
+	jp nz,RetryISRI		; erstmal so lassen
+
+SkipCmpFNam:
 	ld de,(iobuf+17)
 	ld hl,(iobuf+19)
 	or a	
@@ -315,7 +388,8 @@
 	call TestBreak
 	ld e,005h
 	call CCaos
-	jr c,C_MBI
+	call ChkIOErr
+	jr c,C_MBI 		; nur noch bei DEV 0
 	IRM_ON
 	ld e,(ix+002h)
 	IRM_OFF
@@ -355,8 +429,10 @@
 	ldir
 	ld e,00bh
 	call CCaos
+	call ChkIOErr3
 	ld a,012h
 	call OutChr
+	call PrNL
 	ld hl,(l06b6h)
 l04d5h:
 	ret	
@@ -379,6 +455,7 @@
 	ld l,a
 	IRM_ON
 	ld (CAOS_CURSO),hl
+IRMoff_ret:
 	IRM_OFF
 	ret	
 sub_0502h:
@@ -572,6 +649,8 @@
 l06c3h:
 	call SaveCAOS_SP
 	jr Reset
+PasPrgStartA:
+	call IRMoff_ret
 PasPrgStart:
 	jp 07cd5h
 JEndPascal:
@@ -603,7 +682,7 @@
 JCompileToRuntimeEnd:
 	jp SetBinStartToRuntimeEnd
 fileName:
-	defb 000h,000h,000h,000h,000h,000h,000h,000h
+	defb '        '
 fileExt:
 	defb 000h,000h,000h
 	defb 000h
@@ -677,7 +756,7 @@
 	ld hl,fileName
 	push hl	
 SetFNClear:
-	ld (hl),000h
+	ld (hl),020h		; 20h statt 0 wg. ISRI und autom. Dateiendung COM
 	inc hl	
 	djnz SetFNClear
 	pop hl	
@@ -5268,9 +5347,10 @@
 	defb 00dh
 banner:
  	defb 00ch,012h
- 	defb '**** KC-PASCAL V5.1 ****',00dh,00dh
- 	defb 'BEARB. VON  +++ AM90 +++',00dh,00dh
- 	defb '   (VERSION KC85/4)',00dh,000h
+version:
+ 	defb '**** KC-PASCAL V5.1d ****',00dh,00dh
+ 	defb '  BEARB. VON AM90, DU25',00dh,00dh
+ 	defb '   (VERSION KC85/4/5)',00dh,000h
 CoVarInitData:
 	defb 020h
 	defw RL_PACKED
@@ -11389,6 +11469,9 @@
 	defs 13
 
 PasEx:
+	call SetErrDCV		; ruft GetCAOSVer auf
+	cp 046h
+	ret nc			; PasEx wird ab CAOS 4.6 nicht mehr gebraucht
 	ld hl,00000h
 	ld b,080h
 PXSrchProlog:
@@ -11419,7 +11502,8 @@
 	ld bc,0fc80h
 	in a,(c)
 	cp 0a7h
-	jp nz,PXASCI
+	ld c,0
+	jr nz,PXInitDev
 	ld bc,SUBNEU-PXTAPE
 	ldir
 	ld a,07fh
@@ -11431,7 +11515,7 @@
 	ld (SUBALT),hl
 	ld de,SUBNEU
 	ld (SUTAB),de
-	ld bc,00092h
+	ld bc,00092h		; abhängig von der CAOS-Version
 	ldir
 	ld hl,MBO
 	ld (SUBNEU+2),hl
@@ -11441,6 +11525,9 @@
 	ld de,SUBNEU+16
 	ld c,008h
 	ldir
+	ld c,4
+PXInitDev:
+	call PXSetDev
 	jp PXASCI
 PXSTAB:
 	defw ISRO
@@ -11468,14 +11555,21 @@
 	defb 'PASTAPE'
 	defb 001h
 	ld hl,(SUBALT)
+	ld c,0
 PXSetSUTAB:
 	ld (SUTAB),hl
+PXSetDev:
+	ld a,(ix+8)
+	and 0e3h
+	or c
+	ld (ix+8),a
 	ret	
 PXDISK:
 	defw 00000h
 	defb 'PASDISK'
 	defb 001h
 	ld hl,SUBNEU
+	ld c,4
 	jr PXSetSUTAB
 ISRO:
 	ld (ix+002h),000h
